diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/base/status mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/base/status
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/base/status	2006-03-15 23:22:21.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/base/status	2006-11-23 21:23:35.000000000 -0700
@@ -1 +1 @@
-disabled
+enabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/base/Visible mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/base/Visible
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/base/Visible	2006-03-15 23:22:21.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/base/Visible	2006-11-23 21:23:00.000000000 -0700
@@ -1 +1 @@
-no
+yes
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/updates/status mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/updates/status
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/updates/status	2006-03-15 23:22:21.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/updates/status	2006-11-23 21:23:39.000000000 -0700
@@ -1 +1 @@
-disabled
+enabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/updates/Visible mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/updates/Visible
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/updates/Visible	2006-03-15 23:22:21.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/updates/Visible	2006-11-23 21:23:12.000000000 -0700
@@ -1 +1 @@
-no
+yes
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/migrate/20CentOSRepos mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/migrate/20CentOSRepos
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/migrate/20CentOSRepos	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/migrate/20CentOSRepos	2006-11-23 21:25:58.000000000 -0700
@@ -0,0 +1,8 @@
+{
+    for my $name ( qw(base updates) )
+    {
+	my $repo = $DB->get($name) or next;
+
+        $repo->set_prop('Visible', 'yes');
+    }
+}
