diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/yum mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/yum
--- smeserver-yum-1.2.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/yum	2006-03-15 23:22:21.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/yum	2006-11-23 20:15:52.000000000 -0700
@@ -238,7 +238,9 @@
 	       after adding or removing software,
 	       or applying updates. 
 
-	       <p>	
+                <p>
+		Please click on the Reconfigure button at the bottom of
+		this page to start the reconfiguration process.
 	       <b>An automatic reboot will be
 	       initiated as part of this reconfiguration process</b>
 		</p>
