--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/smeextras/EnableGroups.smeextras	2008-09-19 11:09:11.000000000 -0600
+++ smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/smeextras/EnableGroups	2008-09-19 11:04:54.000000000 -0600
@@ -0,0 +1 @@
+yes
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/smeextras/GPGCheck.smeextras	2008-09-19 11:09:11.000000000 -0600
+++ smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/smeextras/GPGCheck	2005-09-25 20:30:54.000000000 -0600
@@ -0,0 +1 @@
+yes
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/smeextras/MirrorList.smeextras	2008-09-19 11:09:11.000000000 -0600
+++ smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/smeextras/MirrorList	2008-09-19 11:04:53.000000000 -0600
@@ -0,0 +1 @@
+file:///etc/yum.repos.d/mirrors-smeextras
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/smeextras/Name.smeextras	2008-09-19 11:09:11.000000000 -0600
+++ smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/smeextras/Name	2005-09-25 20:30:54.000000000 -0600
@@ -0,0 +1 @@
+SME Server - updates
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/smeextras/status.smeextras	2008-09-19 11:09:11.000000000 -0600
+++ smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/smeextras/status	2005-05-27 04:34:42.000000000 -0600
@@ -0,0 +1 @@
+enabled
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/smeextras/type.smeextras	2008-09-19 11:09:11.000000000 -0600
+++ smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/smeextras/type	2005-05-27 04:34:42.000000000 -0600
@@ -0,0 +1 @@
+repository
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/smeextras/Visible.smeextras	2008-09-19 11:09:11.000000000 -0600
+++ smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/smeextras/Visible	2006-03-15 23:22:21.000000000 -0700
@@ -0,0 +1 @@
+yes
