diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-CentOS-5 mezzanine_patched_smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-CentOS-5
--- smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-CentOS-5	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-CentOS-5	2007-07-14 09:37:49.000000000 -0600
@@ -0,0 +1,28 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1.2.6 (GNU/Linux)
+
+mQGiBEWfB6MRBACrnYW6yKMT+MwJlCIhoyTxGf3mAxmnAiDEy6HcYN8rivssVTJk
+CFtQBlBOpLV/OW2YtKrCO2xHn46eNfnMri8FGT8g+9JF3MUVi7kiV1He4iJynHXB
++F2ZqIvHf3IaUj1ys+p8TK64FDFxDQDrGQfIsD/+pkSGx53/877IrvdwjwCguQcr
+Ioip5TH0Fj0OLUY4asYVZH8EAIqFHEqsY+9ziP+2R3/FyxSllKkjwcMLrBug+cYO
+LYDD6eQXE9Mq8XKGFDj9ZB/0+JzK/XQeStheeFG75q3noq5oCPVFO4czuKErIRAB
+qKbDBhaTj3JhOgM12XsUYn+rI6NeMV2ZogoQCC2tWmDETfRpYp2moo53NuFWHbAy
+XjETA/sHEeQT9huHzdi/lebNBj0L8nBGfLN1nSRP1GtvagBvkR4RZ6DTQyl0UzOJ
+RA3ywWlrL9IV9mrpb1Fmn60l2jTMMCc7J6LacmPK906N+FcN/Docj1M4s/4CNanQ
+NhzcFhAFtQL56SNyLTCk1XzhssGZ/jwGnNbU/aaj4wOj0Uef5LRGQ2VudE9TLTUg
+S2V5IChDZW50T1MgNSBPZmZpY2lhbCBTaWduaW5nIEtleSkgPGNlbnRvcy01LWtl
+eUBjZW50b3Mub3JnPohkBBMRAgAkBQJFnwekAhsDBQkSzAMABgsJCAcDAgMVAgMD
+FgIBAh4BAheAAAoJEKikR9zoViiXKlEAmwSoZDvZo+WChcg3s/SpNoWCKhMAAJwI
+E2aXpZVrpsQnInUQWwkdrTiL5YhMBBMRAgAMBQJFnwiSBYMSzAIRAAoJEDjCFhY5
+bKCk0hAAn134bIx3wSbq58E6P6U5RT7Z2Zx4AJ9VxnVkoGHkVIgSdsxHUgRjo27N
+F7kBDQRFnwezEAQA/HnJ5yiozwgtf6jt+kii8iua+WnjqBKomPHOQ8moxbWdv5Ks
+4e1DPhzRqxhshjmub4SuJ93sgMSAF2ayC9t51mSJV33KfzPF2gIahcMqfABe/2hJ
+aMzcQZHrGJCEX6ek8l8SFKou7vICzyajRSIK8gxWKBuQknP/9LKsoczV+xsAAwUD
+/idXPkk4vRRHsCwc6I23fdI0ur52bzEqHiAIswNfO521YgLk2W1xyCLc2aYjc8Ni
+nrMX1tCnEx0/gK7ICyJoWH1Vc7//79sWFtX2EaTO+Q07xjFX4E66WxJlCo9lOjos
+Vk5qc7R+xzLDoLGFtbzaTRQFzf6yr7QTu+BebWLoPwNTiE8EGBECAA8FAkWfB7MC
+GwwFCRLMAwAACgkQqKRH3OhWKJfvvACfbsF1WK193zM7vSc4uq51XsceLwgAoI0/
+9GxdNhGQEAweSlQfhPa3yYXH
+=o/Mx
+-----END PGP PUBLIC KEY BLOCK-----
+
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora-extras mezzanine_patched_smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora-extras
--- smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora-extras	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora-extras	2006-10-15 12:07:02.000000000 -0600
@@ -0,0 +1,35 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1.0.7 (GNU/Linux)
+
+mQGiBEG+vu8RBACyH3cUSp+DXbyIvZ075bGDhjhRCf8m44UW2cCQdO/45CJBkRnC
+AA6SHthCrXCVKIuuo1jSwy8kMXt1P/7IpF6Tc8ZMzcHKTyNetuuD/A7oavgnrJNb
+QVTScqbD6VrvjUAPpDz5TMPqMMFYwylp+ySKMQpoPOYoqdNE3XwnA5vOywCg44+S
+KDWIV0j5xCeIAoa47VmgF2cEAK/FIZRkbb7+i2Fob1mxKsnuLN/xx67aBpU2w7HO
+scVnkke08lRestDVk/6xb+lrq+gVLdNBfAANBrOcwRv2heu+Zu3PZGuC166/JWFM
+kx5a9TkI0zQAw72LqKeX4VDBRz5fUoIs/C+3aGc5Mh+h8fZYjJGf0kr/90rnd/0e
+WqsBA/9ONArdHN44aICt7xku/t98p9Da04A7kRYTHlLrBk5i0U4JJXKei7ZAzNi4
+Or//8ATepQdsBMCJVx1Ss/vJKxupLG9Mw43nbNuA5SmOSPnAoyglYadKyaUXBbVj
+mkqf9ZkjOMfD5u9PEWMWMq3f8uf05qxsrBDeiyc7Fk0jBYTvnrQ4RmVkb3JhIFBy
+ZSBFeHRyYXMgUmVsZWFzZSA8cHJlLWV4dHJhc0BmZWRvcmFwcm9qZWN0Lm9yZz6I
+XgQTEQIAHgUCQb6+7wIbAwYLCQgHAwIDFQIDAxYCAQIeAQIXgAAKCRCC7ZUEGscM
+5uAMAKC+XaPSCYIpdcmJYEe9rNxYMz/hLwCeOyBYUYA2L0WnbfXuNafRqwta/s2I
+RgQSEQIABgUCQgD+/AAKCRC0QmnQTypv0lVYAJoCLolPSuAj873GbV2+m3Bj6sS/
+UACeNcDmIWeruRld83uSQcx4xJyEOUa0KUZlZG9yYSBQcm9qZWN0IDxmZWRvcmEt
+ZXh0cmFzQHJlZGhhdC5jb20+iF4EExECAB4FAkIAeFQCGwMGCwkIBwMCAxUCAwMW
+AgECHgECF4AACgkQgu2VBBrHDOYu8wCffLm57mvcYApV7eqvHXr29gNAbLwAoJVq
+JWvgBIvg8/4MiqwKAQ1MDkIUiEYEEhECAAYFAkIA/wQACgkQtEJp0E8qb9LQJACf
+c0LteS513+5O6erQO7RBUxC/NI4AniDHK+Ogf5l5Gs2ciwD+WApsZ4mPtDBGZWRv
+cmEgUHJvamVjdCA8ZmVkb3JhLWV4dHJhc0BmZWRvcmFwcm9qZWN0Lm9yZz6IXgQT
+EQIAHgUCQgB4iQIbAwYLCQgHAwIDFQIDAxYCAQIeAQIXgAAKCRCC7ZUEGscM5gZ7
+AJ9n8CYN6nvMqxCbQQZwvs7e/J4//QCeM3nDwC5EfEbEtOHPIYAqTf3IQZaIRgQS
+EQIABgUCQgD/BAAKCRC0QmnQTypv0qKuAJ0cnqk9BbbblRSBbn72bK/BAquzywCe
+M7S0C6rfrYlfOjQHQdVB5GitT9u5AQ0EQb6+8RAEAJmsZ5apOFz0Fj6RwX0D+nOU
+Atdzd3zpTNzu4j8+L750ePutjuwizEmCxKycchuG+gGasLmdxY+xk5AUB7gLt4ms
+eKsWyHvQt+H4wG8jjOjHWXlaJkZaYQUxpVPd6/11hkoqpodJ9ckfoIJnpUJWDcd3
+HTW7jdq+ZtBXjQlwvh3HAAMHA/9g50+hqjn23d91SaFTkvaAySZYHEF/xm5sOV1X
+Z37XNFdl7JRl6BN7y59PE9Nk6qP4yHqH1G2YNDL3BmejChMF5GmEqluNdnvfFvwk
+yqPvE7qCv8BhktPqsnrpFmtfeDFI4vztD08oab9LdkDvQwLHBZpc+wCb4VJzOkaZ
+7e7VP4hJBBgRAgAJBQJBvr7xAhsMAAoJEILtlQQaxwzmxdYAn3X4X/ENC3V4yPDR
+7X6oVeIVr5fvAJ4n5vY10NpvYKV5arzKd8DSW2m5Bg==
+=N/fy
+-----END PGP PUBLIC KEY BLOCK-----
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-SMEServer mezzanine_patched_smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-SMEServer
--- smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-SMEServer	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-SMEServer	2007-07-14 09:37:49.000000000 -0600
@@ -0,0 +1,19 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1.2.6 (GNU/Linux)
+
+mQGiBEM84T4RBACMPEsM0PcEqt0hFSOWI4BLLUfBbXZVDekRD24vEq9wGQWWvNxf
+tABh53/eLemGxLF28dqipU5TK+6B0FFDkRyaBppmhmtgBD4jDfw9hPoIIw8ufHO6
+F929yBdRuKLPMn+uQ74Bva1JEhUChxMafE+yyrXORXvZWjR5sc2THJWTywCgnfpQ
+ENzljYR1TcM451xgYxIQKBkD/jHPXM7jfCUYobUQ2P80GqaPCnNtA5nhR9/jyUmP
+KPEgiQCVYQmfXWL6/LnnHEi0IFmZ7bDZc7hLO4JhWQINDgg5WDFNflawCeqZ2KIp
+j+SnwbudnXmE4QdGJ66DrS7eNqJkFvLAT/S2Y2A9QIZCWKrg3HfvbrmiR4abRcy0
+kzBVA/sE/6OTLSO5o/kWBIuPQ616800vhXhLdITEPETkkaA3Oj0POKP6AJi7aLKO
+KqpZzyLZi80vhCauztiQmY6fJiHQym6b/HsRSKf+DjExxoq4zp3TEojUK8ReSY6P
+Mx9nsK5l+T70BX4iJsFdcjJHipnXBcuHOMlQtRlszU0b3NMLE7QvU01FIFNlcnZl
+ciA3IHNpZ25pbmcga2V5IDxidWd0ZWFtQGNvbnRyaWJzLm9yZz6IXgQTEQIAHgUC
+QzzhPgIbAwYLCQgHAwIDFQIDAxYCAQIeAQIXgAAKCRA9cgUIHpyTCJWFAJ4xgIUK
+VQysThX/neAbxWvl2YeqpQCeOOUivZbOSScohei7r8jCa+xUiVWIRgQTEQIABgUC
+QzzjFwAKCRC8R3SEoTuRTTJbAJ9auKPxDeQ1hLN43hO5/bmd+TUKggCeIKAFeaR7
+9Pfl09sv0czvNTFSLRM=
+=z7FN
+-----END PGP PUBLIC KEY BLOCK-----
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-SMEServer7 mezzanine_patched_smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-SMEServer7
--- smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-SMEServer7	2005-10-06 19:28:21.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.2.0/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-SMEServer7	1969-12-31 17:00:00.000000000 -0700
@@ -1,19 +0,0 @@
------BEGIN PGP PUBLIC KEY BLOCK-----
-Version: GnuPG v1.2.6 (GNU/Linux)
-
-mQGiBEM84T4RBACMPEsM0PcEqt0hFSOWI4BLLUfBbXZVDekRD24vEq9wGQWWvNxf
-tABh53/eLemGxLF28dqipU5TK+6B0FFDkRyaBppmhmtgBD4jDfw9hPoIIw8ufHO6
-F929yBdRuKLPMn+uQ74Bva1JEhUChxMafE+yyrXORXvZWjR5sc2THJWTywCgnfpQ
-ENzljYR1TcM451xgYxIQKBkD/jHPXM7jfCUYobUQ2P80GqaPCnNtA5nhR9/jyUmP
-KPEgiQCVYQmfXWL6/LnnHEi0IFmZ7bDZc7hLO4JhWQINDgg5WDFNflawCeqZ2KIp
-j+SnwbudnXmE4QdGJ66DrS7eNqJkFvLAT/S2Y2A9QIZCWKrg3HfvbrmiR4abRcy0
-kzBVA/sE/6OTLSO5o/kWBIuPQ616800vhXhLdITEPETkkaA3Oj0POKP6AJi7aLKO
-KqpZzyLZi80vhCauztiQmY6fJiHQym6b/HsRSKf+DjExxoq4zp3TEojUK8ReSY6P
-Mx9nsK5l+T70BX4iJsFdcjJHipnXBcuHOMlQtRlszU0b3NMLE7QvU01FIFNlcnZl
-ciA3IHNpZ25pbmcga2V5IDxidWd0ZWFtQGNvbnRyaWJzLm9yZz6IXgQTEQIAHgUC
-QzzhPgIbAwYLCQgHAwIDFQIDAxYCAQIeAQIXgAAKCRA9cgUIHpyTCJWFAJ4xgIUK
-VQysThX/neAbxWvl2YeqpQCeOOUivZbOSScohei7r8jCa+xUiVWIRgQTEQIABgUC
-QzzjFwAKCRC8R3SEoTuRTTJbAJ9auKPxDeQ1hLN43hO5/bmd+TUKggCeIKAFeaR7
-9Pfl09sv0czvNTFSLRM=
-=z7FN
------END PGP PUBLIC KEY BLOCK-----
