diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/createlinks mezzanine_patched_smeserver-yum-1.2.0/createlinks
--- smeserver-yum-1.2.0/createlinks	2006-03-16 17:22:21.000000000 +1100
+++ mezzanine_patched_smeserver-yum-1.2.0/createlinks	2006-04-18 16:41:50.324768116 +1000
@@ -15,14 +15,12 @@
     event_link("yum-action", "yum-$_", "20");
     event_link("initialize-default-databases", "yum-$_", "50");
     event_link("navigation-conf", "yum-$_", "80");
-    event_link("yum-update-dbs", "yum-$_", "90");
 }
 
-# for ( qw(install modify remove update) )
-# {
-    # safe_symlink("once",
-	# "root/etc/e-smith/events/yum-$_/services2adjust/yum");
-# }
+for ( qw(install remove update modify) )
+{
+    event_link("yum-update-dbs", "yum-$_", "90");
+}
 
 for ( qw(bootstrap-console-save) )
 {
