diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/templates/etc/yum.conf/10main_debuglevel mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/templates/etc/yum.conf/10main_debuglevel
--- smeserver-yum-1.2.0/root/etc/e-smith/templates/etc/yum.conf/10main_debuglevel	2005-05-27 04:34:42.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/templates/etc/yum.conf/10main_debuglevel	2008-03-14 11:52:07.000000000 -0600
@@ -1 +1 @@
-debuglevel=2
+debuglevel={ $yum{Debug} || '2' }
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/sbin/e-smith/yum mezzanine_patched_smeserver-yum-1.2.0/root/sbin/e-smith/yum
--- smeserver-yum-1.2.0/root/sbin/e-smith/yum	2008-03-14 11:56:37.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.2.0/root/sbin/e-smith/yum	2008-03-14 11:55:51.000000000 -0600
@@ -4,12 +4,12 @@
 echo "WARNING: Additional commands may be required after running yum"
 echo "=============================================================="
 
-BEFORE=$(rpm -qa | md5sum)
+BEFORE=$(rpm -qa 2>&1 | sed '/skipping V4 signature/d' | md5sum)
 
 /usr/bin/yum "$@"
 status=$?
 
-AFTER=$(rpm -qa | md5sum)
+AFTER=$(rpm -qa 2>&1 | sed '/skipping V4 signature/d' | md5sum)
 if [ "$BEFORE" = "$AFTER" ]
 then
   echo "================================================================"
