diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/yum mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/yum
--- smeserver-yum-1.2.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/yum	2008-04-27 18:37:19.000000000 +0200
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/yum	2008-04-27 18:35:18.000000000 +0200
@@ -190,11 +190,6 @@
     </entry>
 
     <entry>
-	<base>REMOVE_SOFTWARE</base>
-	<trans>Remove</trans>
-    </entry>
-
-    <entry>
         <base>DESC_BUTTON_CONFIGURATION</base>
         <trans>
             <![CDATA[
@@ -243,11 +238,6 @@
      </entry>
 
     <entry>
-	<base>POST_UPGRADE</base>
-	<trans>Reconfigure</trans>
-    </entry>
-
-    <entry>
 	<base>ERROR_UPDATING</base>
 	<trans>An error occurred while updating the system.</trans>
     </entry>
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/web/functions/yum mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/web/functions/yum
--- smeserver-yum-1.2.0/root/etc/e-smith/web/functions/yum	2008-04-27 18:37:19.000000000 +0200
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/web/functions/yum	2008-04-27 18:36:34.000000000 +0200
@@ -212,7 +212,7 @@
 
         <subroutine src="print_skip_header()" />
 
-        <subroutine src="print_button('REMOVE_SOFTWARE')" />
+        <subroutine src="print_button('REMOVE')" />
     </page>
 
     <page name="YUM_PAGE_CONFIGURATION" 
@@ -260,7 +260,7 @@
 
         <subroutine src="display_yum_log()" />
 
-        <subroutine src="print_button('POST_UPGRADE')" />
+        <subroutine src="print_button('RECONFIGURE')" />
     </page>
 
 </form>
