--- smeserver-yum-2.0.0/root/etc/e-smith/events/actions/yum-action.buffer	2008-10-07 08:21:29.000000000 -0600
+++ smeserver-yum-2.0.0/root/etc/e-smith/events/actions/yum-action	2008-10-28 16:49:15.000000000 -0600
@@ -40,6 +40,9 @@
 open STDOUT, '>', "$log_file" or die "Can't redirect STDOUT: $!";
 open STDERR, ">&STDOUT"     or die "Can't dup STDOUT: $!";
 
+# Make yum send output immediately, so server-manager sees progress
+$ENV{PYTHONUNBUFFERED} = 1;
+
 # XXX - FIXME - For groupremove, we need to do a grouplist on the group
 # and then work out what to do
 
