diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY.atrpms mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY.atrpms
--- smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY.atrpms	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY.atrpms	2004-12-31 09:42:41.000000000 -0700
@@ -0,0 +1,30 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1.2.6 (GNU/Linux)
+
+mQGiBD5gtCgRBACKIvjMF+20r9k/Uw2Hq6Y/qn1nM0AZEFalhglXP5pMm5bMgkcI
+1vCWqJxSbhQhk8hSEenoszes8hyUxHj4hFFUDiRtAxOpCpGCsCnUddgQtHAQd+tm
+aQsM6J3Jm/EZPtwR0lvwvRGvz2x6Rr95G8+42KK9x+mBYhLk0y3gAbBzhwCgnkDH
+a97MGBT7gRLrmtFqiHrWlPkD/2tBaH6IEuoJhcAbNj9MukbhDOYJ6ic9Nzf6sR3t
+ZG+XgQLLS2DNy8+HWcYJOjpJDEe8zWFDdUv3cL1D0U2f2e85FuJaMucHn+816iw8
+mNjZXJEoDE4LJ8Vv53fkevNZpdWmO2VtRwI+woDnIHYHukDLj2sWhVt+5W+uOKAE
+OippA/9OzuWrwBtTR+Np8ApZGkxhxU1z0iEStV+kQNqJE7YoR4SGMuzEa3bFzrPx
+k4qIU+rw4YgFgHrs1x08lXxNOZkq6avvbl60HqN2qF2UQL/YdU+5X3ixaJVaYYk8
+yuK+hp0Hx2DdBWmVhq6rEzIfpnFhF4qspwMWEiiBGjYDL62W7LQ0QVRycG1zLm5l
+dCAocnBtIHNpZ25pbmcga2V5KSA8QXhlbC5UaGltbUBBVHJwbXMubmV0PohnBBMR
+AgAnAhsDBgsJCAcDAgMVAgMDFgIBAh4BAheAAhkBBQJB1X6uBQkFVf4GAAoJEFCM
+5eZmU0wr0wUAmQF0c6+9GiXCV2WOKtT++GM6YKHnAJ0cDppTgYooRSritXjCG+cr
+Mk8llYhnBBMRAgAnAhsDBQkDdMLsBgsJCAcDAgMVAgMDFgIBAh4BAheABQJAKteu
+AhkBAAoJEFCM5eZmU0wrMMUAnRjS2PXQp0tsC/69IGMMxqU+8xeAAJ9XQjVAo+mU
+kg/3AeBlMBIlFe5hDbQ2QVRycG1zIChycG0gc2lnbmluZyBrZXkpIDxBeGVsLlRo
+aW1tQHBoeXNpay5mdS1iZXJsaW4+iEkEMBECAAkFAkHVfmQCHQAACgkQUIzl5mZT
+TCsESwCcCgs7hLZMF0BThsLvxwnC9qSvHkcAn0UkCAzjm7Q2KsuoX5cS+rNvAgf7
+iEwEEBECAAwFAj5gtJsFgwHhMw0ACgkQQBVS1GOamfHbLgCfWP8SvaGYzOWuJIoV
+TToAkYcF0tEAoIuDT91/XarnY/ZUO+Rolw9w7leNiF8EExECAB8ECwcDAgMVAgMD
+FgIBAh4BAheABQJB1X6uBQkFVf4GAAoJEFCM5eZmU0wrOtEAn0QhRI4QG3vo5zm3
+RZdd2JHwOvw0AJ9UHpxXmlsL0zbdLVcWvobRWLfGgYhfBBMRAgAfBQI+YLQoBQkB
+4TOABAsHAwIDFQIDAxYCAQIeAQIXgAAKCRBQjOXmZlNMKx1jAJwPYAl6NLcSYWeR
+Bz56nWqWv39XBACeP7jdZExNZu+vQjONd+zXsMU8MpuIXwQTEQIAHwQLBwMCAxUC
+AwMWAgECHgECF4AFAj/wTxQFCQN0wuwACgkQUIzl5mZTTCsPOgCdHnJrsLx2shkq
+7IS2f0auH+D+m8wAoIgencfOcNUsuJDLYhaGLjjBKcgw
+=IhnP
+-----END PGP PUBLIC KEY BLOCK-----
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-beta mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-beta
--- smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-beta	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-beta	2005-09-30 01:26:32.059743568 -0600
@@ -0,0 +1,32 @@
+The following public key can be used to verify RPM packages built and
+signed by Red Hat, Inc. for this beta using `rpm -K' using the GNU GPG
+package.  Questions about this key should be sent to security@redhat.com.
+
+
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1.0.6 (GNU/Linux)
+Comment: For info see http://www.gnupg.org
+
+mQGiBDySTqsRBACzc7xuCIp10oj5B2PAV4XzDeVxprv/WTMreSNSK+iC0bEz0IBp
+Vnn++qtyiXfH+bGIE9jqZgIEnpttWhUOaU5LhcLFzy+m8NWfngIFP9QfGmGAe9Gd
+LFeAdhj4RmSG/vgr7vDd83Hz22dv403Ar/sliWO4vDOrMmZBG57WGYTWtwCgkMsi
+UUQuJ6slbzKn82w+bYxOlL0EAIylWJGaTkKOTL5DqVR3ik9aT0Dt3FNVYiuhcKBe
+II4E3KOIVA9kO8in1IZjx2gs6K2UV+GsoAVANdfKL7l9O+k+J8OxhE74oycvYJxW
+QzCgXMZkNcvW5wyXwEMcr6TVd/5BGztcMw8oT3/l2MtAEG/vn1XaWToRSO1XDMDz
++AjUA/4m0mTkN8S4wjzJG8lqN7+quW3UOaiCe8J3SFrrrhE0XbY9cTJI/9nuXHU1
+VjqOSmXQYH2Db7UOroFTBiWhlAedA4O4yuK52AJnvSsHbnJSEmn9rpo5z1Q8F+qI
+mDlzriJdrIrVLeDiUeTlpH3kpG38D7007GhXBV72k1gpMoMcpbQ3UmVkIEhhdCwg
+SW5jLiAoQmV0YSBUZXN0IFNvZnR3YXJlKSA8cmF3aGlkZUByZWRoYXQuY29tPohX
+BBMRAgAXBQI8l5p/BQsHCgMEAxUDAgMWAgECF4AACgkQ/TcmiYl9oHqdeQCfZjw4
+F9sir3XfRAjVe9kYNcQ8hnIAn0WgyT7H5RriWYTOCfauOmd+cAW4iEYEEBECAAYF
+AjyXmqQACgkQIZGAzdtCpg5nDQCfepuRUyuVJvhuQkPWySETYvRw+WoAnjAWhx6q
+0npMx4OE1JGFi8ymKXktuQENBDySTq4QBADKL/mK7S8E3synxISlu7R6fUvu07Oc
+RoX96n0Di6T+BS99hC44XzHjMDhUX2ZzVvYS88EZXoUDDkB/8g7SwZrOJ/QE1zrI
+JmSVciNhSYWwqeT40Evs88ajZUfDiNbS/cSC6oui98iS4vxd7sE7IPY+FSx9vuAR
+xOa9vBnJY/dx0wADBQQAosm+Iltt2uigC6LJzxNOoIdB5r0GqTC1o5sHCeNqXJhU
+ExAG8m74uzMlYVLOpGZi4y4NwwAWvCWC0MWWnnu+LGFy1wKiJKRjhv5F+WkFutY5
+WHV5L44vp9jSIlBCRG+84jheTh8xqhndM9wOfPwWdYYu1vxrB8Tn6kA17PcYfHSI
+RgQYEQIABgUCPJJergAKCRD9NyaJiX2geiCPAJ4nEM4NtI9Uj8lONDk6FU86PmoL
+yACfb68fBd2pWEzLKsOk9imIobHHpzE=
+=gpIn
+-----END PGP PUBLIC KEY BLOCK-----
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY.dag.txt mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY.dag.txt
--- smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY.dag.txt	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY.dag.txt	2003-08-25 04:10:53.000000000 -0600
@@ -0,0 +1,32 @@
+The following public key can be used to verify RPM packages
+downloaded from  http://dag.wieers.com/apt/  using 'rpm -K'
+if you have the GNU GPG package.
+Questions about this key should be sent to:
+Dag Wieers <dag@wieers.com>
+
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1.2.1 (GNU/Linux)
+
+mQGiBD9JMT0RBAC9Q2B0AloUMTxaK73sD0cOu1MMdD8yuDagbMlDtUYA1aGeJVO6
+TV02JLGr67OBY+UkYuC1c3PUwmb3+jakZd5bW1L8E2L705wS0129xQOZPz6J+alF
+5rTzVkiefg8ch1yEcMayK20NdyOmhDGXQXNQS8OJFLTIC6bJs+7MZL83/wCg3cG3
+3q7MWHm3IpJb+6QKpB9YH58D/2WjPDK+7YIky/JbFBT4JPgTSBy611+bLqHA6PXq
+39tzY6un8KDznAMNtm+NAsr6FEG8PHe406+tbgd7tBkecz3HPX8nR5v0JtDT+gzN
+8fM3kAiAzjCHUAFWVAMAZLr5TXuoq4lGTTxvZbwTjZfyjCm7gIieCu8+qnPWh6hm
+30NgA/0ZyEHG6I4rOWqPks4vZuD+wlp5XL8moBXEKfEVOMh2MCNDRGnvVHu1P3eD
+oHOooVMt9sWrGcgxpYuupPNL4Uf6B6smiLlH6D4tEg+qCxC17zABI5572XJTJ170
+JklZJrPGtnkPrrKMamnN9MU4RjGmjh9JZPa7rKjZHyWP/z/CBrQ1RGFnIFdpZWVy
+cyAoRGFnIEFwdCBSZXBvc2l0b3J5IHYxLjApIDxkYWdAd2llZXJzLmNvbT6IWQQT
+EQIAGQUCP0kxPQQLBwMCAxUCAwMWAgECHgECF4AACgkQog5SFGuNeeYvDQCeKHST
+hIq/WzFBXtJOnQkJGSqAoHoAnRtsJVWYmzYKHqzkRx1qAzL18Sd0iEYEEBECAAYF
+Aj9JMWAACgkQoj2iXPqnmevnOACfRQaageMcESHVE1+RSuP3txPUvoEAoJAtOHon
+g+3SzVNSZLn/g7/Ljfw+uQENBD9JMT8QBACj1QzRptL6hbpWl5DdQ2T+3ekEjJGt
+llCwt4Mwt/yOHDhzLe8SzUNyYxTXUL4TPfFvVW9/j8WOkNGvffbs7g84k7a5h/+l
+IJTTlP9V9NruDt1dlrBe+mWF6eCY55OFHjb6nOIkcJwKxRd3nGlWnLsz0ce9Hjrg
+6lMrn0lPsMV6swADBQP9H42sss6mlqnJEFA97Fl3V9s+7UVJoAIA5uSVXxEOwVoh
+Vq7uECQRvWzif6tzOY+vHkUxOBRvD6oIU6tlmuG3WByKyA1d0MTqMr3eWieSYf/L
+n5VA9NuD7NwjFA1kLkoDwfSbsF51LppTMkUggzwgvwE46MB6yyuqAVI1kReAWw+I
+RgQYEQIABgUCP0kxPwAKCRCiDlIUa4155oktAKDAzm9QYbDpk6SrQhkSFy016BjE
+BACeJU1hpElFnUZCL4yKj4EuLnlo8kc=
+=mqUt
+-----END PGP PUBLIC KEY BLOCK-----
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY.dries.txt mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY.dries.txt
--- smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY.dries.txt	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY.dries.txt	2005-09-29 17:41:13.000000000 -0600
@@ -0,0 +1,37 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1.2.3 (GNU/Linux)
+
+mQGiBD6yQwERBADq84yyi2etmzc4DbZsWjQOYlaNkVQtIxdoC+8AqALYCf/bYdJL
+NYbUZSstcHk11MBfWNlZhyOOi2Z7mBEaqdHwEB1RWynHuvDozIDhO4ivVi+WsaLg
+WGnAGIYYQzP1keFfQcmAdHbjYbFyilJr/Z3rIvESlBCFwnTsCujomRbmLwCgt6UN
+mIqdSIaHpXiNmC6CA8ppaRcEAKG6NFcWDRp8J1LmZjyfhcpC6u3qYocWzPAqzsEB
+RU/6RBsk3N9x/U1q2GrdkMnSnOgSQJ09n9/WYdoXUkphLMLFzVvTnEfWFqQQdBLy
+OTO1Ti9xxJPUXi1EIVOCCQpWIQBu69wxAnnvLb9UYubjX7h+OYfzb+ZmKlyTsodv
+cTGnBACuJvxdJbkKrEW4waGZ2QEQTYffghij3XkLPUpDIhVjWA/1Q6CA019Q4h6z
+U64HuTkdlykTYatJGPy3GEN2/C2jQk2Vy+c76tCbSS3zJcwbeqQM5IPIX32+WkE9
+qTlutFf5PyqNSGAYFw0pUx0fi35rEK5w1Fj9zB7Y4NM2gfx5ebQkRHJpZXMgVmVy
+YWNodGVydCA8ZHJpZXNAdWx5c3Npcy5vcmc+iFkEExECABkFAj6yQwEECwcDAgMV
+AgMDFgIBAh4BAheAAAoJEJwUoZwap4SVCSUAoLVb0j/ljkUsooQoIaIakCkAnPd2
+AJ44fYSm5SIIzzOtTa/DrQqkpqrX+IhGBBMRAgAGBQI+skbGAAoJEBTXsfdF5QlB
+LNUAmgOsSMRcR4u+TryTd1taiWBEO9n8AJ92ECdkT/Ar4FUoPUXToZqONQ6nkIhG
+BBARAgAGBQI/mR7AAAoJEPgch3XYR4XPZcYAn2UB/8xS6CXj0PIFXLh0GmHM2SRA
+AKCNro2OmA7GlcMxjPC325GxTPnL94hGBBIRAgAGBQI+skYLAAoJENBLOZvTzT0y
+lAAAn0uhho8DZAX8PcZ1+7yOqG7gL72aAJ4kvW2uCVze4apiXSyjao2q+17SMIhG
+BBIRAgAGBQI+skYgAAoJEFFr21yLP3p1kdIAn0hEq/2+TpQ7kFEHAiblMGlYIaXh
+AJ9Koz5hae7zBHZ81iwakW1i843h64hGBBIRAgAGBQI+skY2AAoJENO2qfx8lY3F
+GOMAn3wxHgwNM5ReUPFj8Wn8qLiDT2I6AJ9G/3mVy9vFjKqDmWavTEdXsgaqOrkC
+DQQ+skMwEAgA4pdpL1cmYbRG+T72mpnDhbpmcUstS2DIVrGVPmQr5+rT8oiqBCxa
+ymJmvyxdMvq35qzC0QjserbNTVImItlDqBmt61Va70OzZbWC/oPx1fV/SHj6ffdJ
+0mkZS8PCB6EdyDWMxCPraDtJot7vbQMR9UKxRcvKHUWWUfm6RfJKDkzvWY+vLDoG
+Jd1dDZKn/RHnhmZDjvjzJjlQO73tumMcMjCWHPDqaWkZchxuzKxHyXleItCJlY5c
+EelNO5o34ELsUZ7vEXp0tPLB2B6XFlDKnrw6HLJlsDfD3Jcs607O9iqGOPzUHQgT
+IMHF+BasndSpIAZkFN9wKzMFO5RGWvLHcwADBgf/UwR4Lzt9yHaY3XbwtKN3ByZy
+TTjlWrfOAsJfj9GYmYcWKAx/pGpJeiLTfeqNlNiQW4eGU2flg25P6LAGvDQK0pc4
+FdH3Ekoz9mIhCbUT8HCFnSekUiliYOR18b7C86+lPRwwTAYtfLELcirWYmJdeZAo
+PhloauK6omapT9UmkhrMiczNRBa45jX4b1n2WJhtt1p9zdO9U8nML/nvGtrnsEOG
+CbdNzSIMtCPA2lbEYjS7asxQqkuhvCbM0MGlffbr3Q4/O980jiGvKGqf7XrBiWap
+llJ4S2d7GgFx3gULF6QJxPbces0ybxS8YAnVw8L0X8zAstV6Q5cFFbz/aWl2VYhG
+BBgRAgAGBQI+skMwAAoJEJwUoZwap4SVMPkAnAspIpPzR97FkNeZ0cP+QGwf77Uj
+AKC3gw9bqWdogkjHwH9iQAAwmy9suw==
+=v9Fn
+-----END PGP PUBLIC KEY BLOCK-----
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora
--- smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora	2005-09-30 01:26:32.060743416 -0600
@@ -0,0 +1,27 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1.0.7 (GNU/Linux)
+
+mQGiBD+dnTsRBACwnlz4AhctOLlVBAsq+RaU82nb5P3bD1YJJpsAce1Ckd2sBUOJ
+D11NUCqH8c7EctOquOZ5zTcWxHiWWbLyKQwUw2SUvnWa5SSbi8kI8q9MTPsPvhwt
+gMrQMLenMO+nsrxrSaG6XcD+ssfJNxC7NQVCQAj3pvvg9rKi3ygsM7CXHwCghgsq
+X6TOr55HE90DbEsoq3b/jjsD/i8aIZ6urUgrpAkQslcakXdJLKgSdwjRUgVZgvYZ
+b7kAx1iPq0t/AhB3NJw3zW4AAKJohGg3xj5K4V8PJEZrSIpoRYlF43Kqlfu2p5gh
+WT89SP4YAlWPeTqf0+dTYUYz3b144k2ZFOdRuXIRxunoYNAUr9oMrxBXbJ/eY+0U
+QX3pBACYzKizyY4JJgd0zFJmNkcdK9nzcm+btYFnYQo33w5GSE686UNr+9yiXt9t
+mPRvNEbj3u+xoAX8B/5k3aZ5NbUhV64/VcKlUdRIxNlFCG7I9KgxeHWAYwi7yqOG
+XM3T/v6o7GLdQEB0ChFqS7kUlqmwLV+C3QhlrFe/Cuk26i+Q6rQiRmVkb3JhIFBy
+b2plY3QgPGZlZG9yYUByZWRoYXQuY29tPohbBBMRAgAbBQI/nZ07BgsJCAcDAgMV
+AgMDFgIBAh4BAheAAAoJELRCadBPKm/S2PAAnRTlhorITphab+oxAHtbxZF9BVyD
+AJ9WOVaZUG53IWWIAXOGv3j/cmr3lohGBBMRAgAGBQI/nZ22AAoJECGRgM3bQqYO
+R5QAoIp1G+omVktq/snxpmz5UeHjlSYjAKCRr/ea/L7S7ZTxB18cf1TYfad1x4hG
+BBARAgAGBQI/ntjgAAoJECnVuiSN9W0FUSUAoJnrone4J0o1HMkRz+6g9KVuO2Fy
+AJ0XyebOzVmI9U5OyOfnNmYV0wnQcrkBDQQ/nZ08EAQAugOfLWJbKwMA9vg2mJU5
+94TZU0HRJkx/fqYhx0YxWWRpzplrEyvcDXuYcWi1Hwh0tD86T4fR5GV6joWiWClz
+D+Hwhhb6gcSdeSGlGLlZAvWYtFSHWiv+3LaI9w8Vtczl99Bh2WiMDNDDGw0RQg6Z
+aftldLSe4j1pffpFGQ8SuisAAwUEAKVxqLT7fC5xQ6oclcZ+PhoDlePQ1BiTS7tu
+GM07bFF4nNvY91LL7S31pooz3XbGSWP8jxzSv1Fw35YhSmWGOBOEXluqMbVQGJJ5
+m8fqJOjC0imbfeWgr/T7zLrJeiljDxvX+6TyawyWQngF6v1Hq6FRV0O0bOp9Npt5
+zqCbDGs/iEYEGBECAAYFAj+dnTwACgkQtEJp0E8qb9L//gCcDVYnDegNCOxDn1se
+dDwxw+0h8OcAn1CZHof15QqxnTwEnvwF2QeOI5dn
+=mJAx
+-----END PGP PUBLIC KEY BLOCK-----
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora-rawhide mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora-rawhide
--- smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora-rawhide	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora-rawhide	2005-09-30 01:26:32.061743264 -0600
@@ -0,0 +1,21 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1.0.7 (GNU/Linux)
+
+mQGiBD+doUwRBADudl/P8RqkzaHstvwbT93MByo7yX+xHcr0FaVAoG9Qfn+2O0qy
+nbNIz9BoieCx9zfgruBBaYjcT1p7QjdyfvF3wLq6ak84ne+QW91rctzqG8JqPhSB
+e34L7T0JnsUdiLfnfq895Hy67SYTOcJ6sN1Eg/eKYPP0CHKL9BY9atGRZwCg14Kh
+va6OSfIDyV6DNnfYF+cUSwsEALARvclTS8lLL+4sgZthg4D4Od4mYayr0QpKpoBI
+0RxOj26v038unOGqEjjqX/F36PwQv6wt6DmaGd9atQ/VUqW4cnKmNfaxJVAqznqW
+q80FKzTPH/JC/20v/5xx0FoxEVSCoKs2YPdXEQfEUAzNG9JC4GHNCxt2Gtnt4o5l
+sn15BADepds5sa/ptu//hJcYhrvcgzUJQR/JQdDbtK+Rzrvi2oZQFZUHzCVVxxfE
+0AULmX5bEC08nAGtp418ceeo1PdqlcVE+kaxuFpPgto/Sd1FvJW4P2B1a7rcUjGc
+BarOMVY8QpFZs5iIlkzZdPns1BCijWlhltOiyrdxyNrM49hjlrRGRmVkb3JhIFBy
+b2plY3QgYXV0b21hdGVkIGJ1aWxkIHNpZ25pbmcga2V5ICgyMDAzKSA8cmF3aGlk
+ZUByZWRoYXQuY29tPohbBBMRAgAbBQI/naFMBgsJCAcDAgMVAgMDFgIBAh4BAheA
+AAoJEOE4XU4c3byppW8An385Vt7LXSAIHqdH85QTCH512cr+AJ9vB4inXB6lxOoq
+sbnKY0HG4Sr1IIhGBBMRAgAGBQI/naOPAAoJELRCadBPKm/SFMgAn2/FfBhf2Yov
+j3gH8Z1t+bbsHRkLAJ94tjdweTZmfZsQZ7YaDXpeJ2iY+4hGBBARAgAGBQI/ntkG
+AAoJECnVuiSN9W0Fi5gAn3wSpemY+8rq8aFfRLEumrSksfUoAJ95YyaGdRXv+fWt
+KfPJSNTqsqhiKA==
+=umuG
+-----END PGP PUBLIC KEY BLOCK-----
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora-test mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora-test
--- smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora-test	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-fedora-test	2005-09-30 01:26:32.062743112 -0600
@@ -0,0 +1,20 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1.0.7 (GNU/Linux)
+
+mQGiBD+do/cRBACAF7Gv91v32uUrT4nZe+qKgLl59HmtvmaOzT4lInrF95Sget7p
+2TpnIqh5H0x0VGpdoeNfHvlXCw+aP50mTJv91BYmLfelkTzokqpEU5YpqGOqa5BV
+jelnoK1VKOcXMB28d79Gz9tn7friZN4SnyWbcSIA4C3dHNhsjifIYf0rywCg3y19
+3N6jODa0VSGeeEb4whE2xccD/i/rqk0MQmW6td2Dz2jTBaHxyf7FIcH18YDtwraD
+otBRxqGkTVyFGc1q7ZTrRsbzdjIfxbp2C6GT2VJ5kHuRm9jf7JdW5fOhun+oLimR
+erTlrPmgz8qQRcqUm6fkmUsYufL1u7DdyfxX7jI2ngkbAR+CsZ8Cj5JhBMeAXc/D
+WWhPA/4mzZ70hn1OrGLokzxNr4O7u1+n0thsxGN0qfuTNGIc09UKCoAN42he7Qjs
+kIqiE7gLtLzx8elxVOK19tNcaAiLoqmtxlLQqB03pLy4V0tMP9QcWQETbc67qTd1
+zASiefGjf364j4lQd+X9AqdyhUrnax3UgPD5IxilBp92Cw7Qi7QzRmVkb3JhIFBy
+b2plY3QgKFRlc3QgU29mdHdhcmUpIDxyYXdoaWRlQHJlZGhhdC5jb20+iFsEExEC
+ABsFAj+do/cGCwkIBwMCAxUCAwMWAgECHgECF4AACgkQ2oTL1DDJ7Pgh8QCgqNNs
+b7xdW+UhzyZw+DEAQF6e+q4AmwVus4u9REyd/aigK2RvrdtcsNy0iEYEExECAAYF
+Aj+dpBoACgkQtEJp0E8qb9Ji4QCfanY0CXnRwtBnm8GEMRNqgfGgcc0AnRunlejp
+JRxGPaRsmY/11stYll32iEYEEBECAAYFAj+e2SIACgkQKdW6JI31bQWWNQCfTKcX
+/7lerpwIVHcAILDduz9qxssAn1t46ERJ9EqsxJxSFlN7oBPyNkt6
+=R9He
+-----END PGP PUBLIC KEY BLOCK-----
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-freshrpms mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-freshrpms
--- smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-freshrpms	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-freshrpms	2005-04-08 03:48:20.000000000 -0600
@@ -0,0 +1,32 @@
+The following public key can be used to verify RPM packages
+downloaded from http://freshrpms.net/ using `rpm -K' if you
+have the GNU GPG package.
+Questions about this key should be sent to :
+Matthias Saou <matthias (at) rpmforge.net>
+
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1.0.4 (GNU/Linux)
+Comment: For info see http://www.gnupg.org
+
+mQGiBDlgvfERBADevsErSu+DAnE90dFPnpEX20elyZAmuRExGbcUJWSoJynohtGC
+a9fW6JY8zm/Dm2dC8f1sSNQ2w7CFG7XRBfyQcL4AqrNKbOMeCl66Tgj+NURUHsnV
+yU3ASXROxVQ4/yJ69PVFomj0sealdEyDXDQoXhvgyI26qe3rriKBefCbRwCg+TdH
+q5I8B/9X7QLnWg7sZk7iI7sD/27S9r4PS+FU9N26BvdgOvGeW6+1O/oqAU6HB+EF
+eGw0+uNbLjxPy1A9R5+M/FBZBMeyTO0S8i4hYKnGbeESfUWdFom7kc0DOEF3bLlf
+NOe+6rsqH2DPOOVbPfcJdOjBTldR7ETx/IyYLWSwdNx71kl/C8GTKKtw31BPUOf7
+sZ7FA/9aDy9sG5Km7fJBGKx66Z3Tmu9bH7oNDesm7uQ8+45qO+njOjFvQ6CfAzzV
+MjzKEm0LSIySaKttDSU9AnlxNypbH9gCx1FwcPcip6vgV10D6YRG8n4cl0LB6Ins
+DoR/MbAv4c8ROw/xiNVYrukxoX7EVpJuNQaP+UoqS9waVfqOh7Q6TWF0dGhpYXMg
+U2FvdSAoVGhpYXMpIDxtYXR0aGlhcy5zYW91QGVzdC51bmUubWFybW90dGUubmV0
+PohWBBMRAgAWBQI5YL3xBAsKBAMDFQMCAxYCAQIXgAAKCRBpKsRZ5C1Ue2rvAKDa
+sk63CmO+/L8cq7HdI98AheFMBwCgwjgUeOCQTVHIlMypOLLvnni7tEa5AQ0EOWC9
+9hAEAMa6uFwa3AZVJW3kTkeoijTmOGkTo15fOivfuyfV5o8JdTc9Fxs/yFV6YVgx
+iKTSUQkPti33irv19bMx5m5nHYccftkagh3y+lDXj4TVrFhgMeucZndXMqREsmbN
+oy3vBdKx/YChWuhRrnhye5ZBztYWgEgMtrIC0F+slHWIdTaTAAMFBAC1g46HPYSA
+UplLYRdd5oTXWG7SoiCvu4wLmNOkVclIK1tepMVFAKhhu3m5w1EiYdfMYW4sRFWA
++z9YyzmWeoJwBkbkmWgR8MHvf5Xp56GSKlB93artFRPuad45vVnoFuwX6yZeSj1i
+TbXFe+YxVm/YoqJkegLMX2HU/C4u2rOcoIhGBBgRAgAGBQI5YL32AAoJEGkqxFnk
+LVR7G4UAoLkksoQD45ntA5PUkJbIGXaEqokPAKCa3RVrnNfp5qzfAn8uo4EWg8eb
+1w==
+=Sm+P
+-----END PGP PUBLIC KEY BLOCK-----
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-rawhide mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-rawhide
--- smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-rawhide	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-rawhide	2005-09-30 01:26:32.063742960 -0600
@@ -0,0 +1,24 @@
+By using `rpm -K' and the GNU GPG package, the following public key can be
+used to verify RPM packages built and signed by Red Hat, Inc.  for rawhide
+and testing.  Questions about this key should be sent to
+security@redhat.com.
+
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1.0.7 (GNU/Linux)
+
+mQGiBD9DmVMRBACW9LRPUtKnUTNeSQN5huiPA0+d0hGLTEG1kkUj47zWC3RXZ1Xh
+c/6yzWOrPk85Q7ys/dGoLLqsfekkxDVipUE0iS+47VMYgm531x4cd2oKnQb3Z/2R
+3A6kjDkc2KyTmfOsZ8DTdL76u81Rnv6MCQjXTJCFOQzv3yxmn6ZDhOoSkwCg24J2
+Qh0zuvvtyfNMsj+S8ksase8D/1qQgcURZDYZkaxrQSsdyZWsqb4NT51snXG5wE4L
+Yh1VGH26YU/kY7YxU5TPo8kbQTztkqaoFHk/O6zx+Q/iS8Izv2ByTsZv0F/Weym7
+Fyf+x+PTA7oDvV9hN/aJy/UPF+t4GRrsEV2mH8Ee3hIZGKSWwTwZZh+JyfIcqkyU
+zwY9BACLOZ8xZjHzU4MjZw2qgWemr96Zyr2foGl0+2G1STnluWf24//nNWBiLFuA
+txucItsppnUM0qB5RZ69N4c9MOmCZkBoahP3Q30qAsaAN4RaaE6AG7Yms0wtPW3m
+qzzhcuCtRAumXqA/oQWYqmHaEwUPenJaxut5rasApm2r5QxaQbRFUmVkIEhhdCwg
+SW5jLiBhdXRvbWF0ZWQgYnVpbGQgc2lnbmluZyBrZXkgKDIwMDMpIDxyYXdoaWRl
+QHJlZGhhdC5jb20+iFsEExECABsFAj9DmVMGCwkIBwMCAxUCAwMWAgECHgECF4AA
+CgkQlM1XQuQY46oixwCfUKmLRHSIBd1FbGTaR5ARBfSsK+MAn1BfxKyahkQ0ScN7
+9RNaYei9kxGIiEYEExECAAYFAj9DvtMACgkQIZGAzdtCpg5exACeJlj9DExab5AK
+C0eQHr8td+WRJ0QAnRNDGXQCBENRgFNgcecDMAr7P6se
+=LmwA
+-----END PGP PUBLIC KEY BLOCK-----
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-SMEServer7 mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-SMEServer7
--- smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-SMEServer7	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.1/root/usr/share/rpm-gpg-keys/RPM-GPG-KEY-SMEServer7	2005-09-30 01:15:26.688895312 -0600
@@ -0,0 +1,19 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
+Version: GnuPG v1.2.6 (GNU/Linux)
+
+mQGiBEM84T4RBACMPEsM0PcEqt0hFSOWI4BLLUfBbXZVDekRD24vEq9wGQWWvNxf
+tABh53/eLemGxLF28dqipU5TK+6B0FFDkRyaBppmhmtgBD4jDfw9hPoIIw8ufHO6
+F929yBdRuKLPMn+uQ74Bva1JEhUChxMafE+yyrXORXvZWjR5sc2THJWTywCgnfpQ
+ENzljYR1TcM451xgYxIQKBkD/jHPXM7jfCUYobUQ2P80GqaPCnNtA5nhR9/jyUmP
+KPEgiQCVYQmfXWL6/LnnHEi0IFmZ7bDZc7hLO4JhWQINDgg5WDFNflawCeqZ2KIp
+j+SnwbudnXmE4QdGJ66DrS7eNqJkFvLAT/S2Y2A9QIZCWKrg3HfvbrmiR4abRcy0
+kzBVA/sE/6OTLSO5o/kWBIuPQ616800vhXhLdITEPETkkaA3Oj0POKP6AJi7aLKO
+KqpZzyLZi80vhCauztiQmY6fJiHQym6b/HsRSKf+DjExxoq4zp3TEojUK8ReSY6P
+Mx9nsK5l+T70BX4iJsFdcjJHipnXBcuHOMlQtRlszU0b3NMLE7QvU01FIFNlcnZl
+ciA3IHNpZ25pbmcga2V5IDxidWd0ZWFtQGNvbnRyaWJzLm9yZz6IXgQTEQIAHgUC
+QzzhPgIbAwYLCQgHAwIDFQIDAxYCAQIeAQIXgAAKCRA9cgUIHpyTCJWFAJ4xgIUK
+VQysThX/neAbxWvl2YeqpQCeOOUivZbOSScohei7r8jCa+xUiVWIRgQTEQIABgUC
+QzzjFwAKCRC8R3SEoTuRTTJbAJ9auKPxDeQ1hLN43hO5/bmd+TUKggCeIKAFeaR7
+9Pfl09sv0czvNTFSLRM=
+=z7FN
+-----END PGP PUBLIC KEY BLOCK-----
