diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/cron.daily/smeserver-yum mezzanine_patched_smeserver-yum-1.2.0/root/etc/cron.daily/smeserver-yum
--- smeserver-yum-1.2.0/root/etc/cron.daily/smeserver-yum	2005-09-25 20:30:54.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/cron.daily/smeserver-yum	2007-02-16 13:29:12.000000000 -0700
@@ -23,5 +23,5 @@
 
 [ $yum_status = 'enabled' ] || exit 0
 
-/usr/bin/runsvctrl o /service/yum
+/usr/bin/sv o /service/yum
 
