diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.2/root/etc/e-smith/web/functions/yum mezzanine_patched_smeserver-yum-1.1.2/root/etc/e-smith/web/functions/yum
--- smeserver-yum-1.1.2/root/etc/e-smith/web/functions/yum	2005-09-26 12:30:54.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.2/root/etc/e-smith/web/functions/yum	2005-10-14 14:41:34.425116931 +1000
@@ -127,7 +127,7 @@
         <field
             type="select" multiple="1"
             id="SelectedPackages"
-            value="get_options('updates','package')"
+            value="get_names('updates','package')"
             options="get_options('updates','package')">
             <label>LABEL_AVAILABLE_UPDATES</label>
             <description>DESC_AVAILABLE_UPDATES</description>
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm mezzanine_patched_smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm
--- smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm	2005-09-26 12:30:54.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm	2005-10-14 14:40:58.919308390 +1000
@@ -150,6 +150,11 @@
     return \%options;
 }
 
+sub get_names
+{
+    return [ keys %{get_options(@_)} ];
+}
+
 sub get_avail
 {
     my ($fm, $yumdb, $type) = @_;
