diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/cron.daily/smeserver-yum mezzanine_patched_smeserver-yum-1.1.0/root/etc/cron.daily/smeserver-yum
--- smeserver-yum-1.1.0/root/etc/cron.daily/smeserver-yum	2005-08-24 20:04:08.501709800 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/cron.daily/smeserver-yum	2005-08-24 20:03:43.067576376 -0600
@@ -19,7 +19,7 @@
 #----------------------------------------------------------------------
 
 yum_status=$(/sbin/e-smith/db configuration getprop yum status)
-yum_status=${status:-disabled}
+yum_status=${yum_status:-disabled}
 
 [ $yum_status = 'enabled' ] || exit 0
 
