diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/yum mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/yum
--- smeserver-yum-1.1.0/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/yum	2005-09-25 15:08:42.963288760 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/yum	2005-09-25 15:08:26.492792656 -0600
@@ -122,8 +122,10 @@
     <entry>
 	<base>HEADER_AVAILABLE_SOFTWARE</base>
 	<trans>
+            <![CDATA[
 	    Les mises à jour suivantes sont disponibles pour &ecirc;tre install&eacute;es 
 	    sur votre système. 
+	    ]]>
 	</trans>
     </entry>
 
@@ -161,8 +163,10 @@
     <entry>
 	<base>HEADER_INSTALLED_SOFTWARE</base>
 	<trans>
+            <![CDATA[
 	    Les mises à jour suivantes sont installées 
 	    sur ce syst&egrave;me. 
+	    ]]>
 	</trans>
     </entry>
 
