diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/addons/MirrorList mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/addons/MirrorList
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/addons/MirrorList	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/addons/MirrorList	2006-11-09 18:45:27.047946907 +1100
@@ -0,0 +1 @@
+http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=addons
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/base/MirrorList mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/base/MirrorList
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/base/MirrorList	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/base/MirrorList	2006-11-09 18:46:06.631052592 +1100
@@ -0,0 +1 @@
+http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=os
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/centosplus/MirrorList mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/centosplus/MirrorList
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/centosplus/MirrorList	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/centosplus/MirrorList	2006-11-09 18:45:27.048946732 +1100
@@ -0,0 +1 @@
+http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=centosplus
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/MirrorList mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/MirrorList
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/MirrorList	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/MirrorList	2006-11-09 18:45:27.048946732 +1100
@@ -0,0 +1 @@
+http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=contrib
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/extras/MirrorList mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/extras/MirrorList
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/extras/MirrorList	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/extras/MirrorList	2006-11-09 18:45:27.048946732 +1100
@@ -0,0 +1 @@
+http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=extras
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/updates/MirrorList mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/updates/MirrorList
--- smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/updates/MirrorList	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/db/yum_repositories/defaults/updates/MirrorList	2006-11-09 18:45:27.047946907 +1100
@@ -0,0 +1 @@
+http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=updates
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/root/etc/e-smith/templates/etc/yum.conf/20repositories mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/templates/etc/yum.conf/20repositories
--- smeserver-yum-1.2.0/root/etc/e-smith/templates/etc/yum.conf/20repositories	2006-03-16 17:22:21.000000000 +1100
+++ mezzanine_patched_smeserver-yum-1.2.0/root/etc/e-smith/templates/etc/yum.conf/20repositories	2006-11-09 18:42:47.557725821 +1100
@@ -13,7 +13,7 @@
         $OUT .= "enabled=" . (($repos->prop('status') eq 'enabled') ? 1 : 0)
 			 . "\n";
 
-	for my $prop (qw(Name BaseURL))
+	for my $prop (qw(Name BaseURL MirrorList))
 	{
 	    $OUT .= lc($prop) . '=' . $props{$prop} . "\n";
 	}
