diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/yum mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/yum
--- smeserver-yum-1.1.0/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/yum	2005-09-23 06:06:07.332089960 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/yum	2005-09-23 06:05:56.310765456 -0600
@@ -67,7 +67,7 @@
 	<trans>
 		La mise à jour logicielle peut installer des logiciels
 		à partir de n'importe quelle site de téléchargement. Pour activer
-		un site de téléchargement, selectionnez le dans la liste. 
+		un site de téléchargement, sélectionnez le dans la liste. 
 		Pour désactiver un site de téléchargement, désélectionnez le dans la liste.
 		Par défaut, seule les version stables, des logiciels testés sont disponibles
 		pour la mise à jour.
