diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/createlinks mezzanine_patched_smeserver-yum-1.1.0/createlinks
--- smeserver-yum-1.1.0/createlinks	2005-07-13 23:38:34.671770128 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/createlinks	2005-07-13 23:38:21.491773792 -0600
@@ -4,15 +4,6 @@
 use File::Basename;
 use File::Path;
 
-for ( qw(yum-arch-repository yum-check-repository yum-update-dbs) )
-{
-    templates2events("/etc/cron.daily/$_", 
-			qw(
-				yum-modify
-				bootstrap-console-save
-			));
-}
-
 templates2events("/etc/yum.conf",
 			qw(
 				yum-modify
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/cron.daily/smeserver-yum mezzanine_patched_smeserver-yum-1.1.0/root/etc/cron.daily/smeserver-yum
--- smeserver-yum-1.1.0/root/etc/cron.daily/smeserver-yum	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/cron.daily/smeserver-yum	2005-07-13 23:36:37.749787640 -0600
@@ -0,0 +1,27 @@
+#!/bin/sh
+#----------------------------------------------------------------------
+# smeserver-yum daily tasks
+# Copyright (C) 2005 Gordon Rowell <gordonr@gormand.com.au>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License or more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307  USA
+#----------------------------------------------------------------------
+
+yum_status=$(/sbin/e-smith/db configuration getprop yum status)
+yum_status=${status:-disabled}
+
+[ $yum_status = 'enabled' ] || exit 0
+
+/usr/bin/runsvctrl o /service/yum
+
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/cron.daily/yum-arch-repository mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/cron.daily/yum-arch-repository
--- smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/cron.daily/yum-arch-repository	2005-05-27 04:34:42.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/cron.daily/yum-arch-repository	1969-12-31 17:00:00.000000000 -0700
@@ -1,20 +0,0 @@
-{
-    return "exit 0  # yum disabled" unless ($yum{status} eq "enabled");
-
-    return "exit 0  # ArchRepository disabled" 
-	unless (($yum{ArchRepository} || "disabled") eq "enabled");
-
-my $Archive = ${'yum'}{'Archive'} || return "# yum{Archive} empty";
-my @Archive = sort(split(/,/ , $Archive));
-
-$OUT = "";
-
-if ($Archive)
-    {
-    foreach $Archive (@Archive)
-        {
-        my @ArchiveURL = split(/\+/, $Archive);
-	$OUT .= "/usr/bin/yum-arch $ArchiveURL[1] > /dev/null 2>&1\n";
-        }
-    }
-}
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/cron.daily/yum-check-repository mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/cron.daily/yum-check-repository
--- smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/cron.daily/yum-check-repository	2005-05-27 04:34:42.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/cron.daily/yum-check-repository	1969-12-31 17:00:00.000000000 -0700
@@ -1,38 +0,0 @@
-{
-    return "exit 0  # yum disabled" unless ($yum{status} eq "enabled");
-
-   return "exit 0  # CheckRepository disabled"
-        unless (($yum{CheckRepository} || "disabled") eq "enabled");
-
-    $OUT = <<'HERE';
-
-#!/bin/sh
-
-# Set QMAIL variable
-export QMAILUSER=root
-
-# Set LOG variable
-LOG=/var/log/yum/yum-check-repository.log
-
-# Create yum-list-updates report
-/bin/echo "`/bin/date`"  > $LOG
-/bin/echo ""  >> $LOG
-/bin/echo "The following rpms are available for updating your server:"  >> $LOG
-/bin/echo ""  >> $LOG
-/usr/bin/yum list updates 2>&1 >> $LOG
-/bin/echo "" >> $LOG
-
-# Create yum-list-available report
-/bin/echo "The following rpms are available for installing on your server:"  >> $LOG
-/bin/echo ""  >> $LOG
-/usr/bin/yum list available 2>&1 >> $LOG
-/bin/echo "" >> $LOG
-
-if [ "$1" != "--nomail" ]; then
-/bin/mail -s "Update Manager Report" admin < $LOG
-fi
-
-exit 0
-
-HERE
-}
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/cron.daily/yum-update-dbs mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/cron.daily/yum-update-dbs
--- smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/cron.daily/yum-update-dbs	2005-07-13 23:38:33.868892184 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/cron.daily/yum-update-dbs	1969-12-31 17:00:00.000000000 -0700
@@ -1,5 +0,0 @@
-{
-    return "exit 0  # yum disabled" unless ($yum{status} eq "enabled");
-
-    return "/usr/bin/runsvctrl o /service/yum";
-}
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/templates.metadata/etc/cron.daily/yum-arch-repository mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates.metadata/etc/cron.daily/yum-arch-repository
--- smeserver-yum-1.1.0/root/etc/e-smith/templates.metadata/etc/cron.daily/yum-arch-repository	2005-05-27 04:34:42.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates.metadata/etc/cron.daily/yum-arch-repository	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-PERMS=0700
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/templates.metadata/etc/cron.daily/yum-check-repository mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates.metadata/etc/cron.daily/yum-check-repository
--- smeserver-yum-1.1.0/root/etc/e-smith/templates.metadata/etc/cron.daily/yum-check-repository	2005-05-27 04:34:42.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates.metadata/etc/cron.daily/yum-check-repository	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-PERMS=0700
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/templates.metadata/etc/cron.daily/yum-update-dbs mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates.metadata/etc/cron.daily/yum-update-dbs
--- smeserver-yum-1.1.0/root/etc/e-smith/templates.metadata/etc/cron.daily/yum-update-dbs	2005-05-27 04:34:42.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates.metadata/etc/cron.daily/yum-update-dbs	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-PERMS=0700
