diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/sbin/e-smith/yum_update_dbs mezzanine_patched_smeserver-yum-1.1.0/root/sbin/e-smith/yum_update_dbs
--- smeserver-yum-1.1.0/root/sbin/e-smith/yum_update_dbs	2005-06-03 17:36:03.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/sbin/e-smith/yum_update_dbs	2005-06-03 17:35:41.000000000 +1000
@@ -27,7 +27,8 @@
 
 if ($db->get_prop('yum', 'AutoInstallUpdates') eq 'enabled')
 {
-    system('/bin/touch', $yum_flag) or warn "Couldn't touch $yum_flag\n";
+    (system('/bin/touch', $yum_flag) == 0) or 
+	warn "Couldn't touch $yum_flag\n";
 }
 else
 {
