diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.2/root/etc/e-smith/db/configuration/force/yum/AutoInstallUpdates mezzanine_patched_smeserver-yum-1.1.2/root/etc/e-smith/db/configuration/force/yum/AutoInstallUpdates
--- smeserver-yum-1.1.2/root/etc/e-smith/db/configuration/force/yum/AutoInstallUpdates	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.2/root/etc/e-smith/db/configuration/force/yum/AutoInstallUpdates	2006-01-24 17:18:49.369308911 +1100
@@ -0,0 +1 @@
+disabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.2/root/etc/e-smith/web/functions/yum mezzanine_patched_smeserver-yum-1.1.2/root/etc/e-smith/web/functions/yum
--- smeserver-yum-1.1.2/root/etc/e-smith/web/functions/yum	2006-01-24 17:20:19.193680480 +1100
+++ mezzanine_patched_smeserver-yum-1.1.2/root/etc/e-smith/web/functions/yum	2006-01-24 17:18:06.354792954 +1100
@@ -97,13 +97,6 @@
             <label>LABEL_YUM_STATUS</label>
         </field>
 
-        <field 
-            type="literal" 
-            id="state_autoinstallupdates" 
-            value="get_status('AutoInstallUpdates', 'localise')">
-            <label>LABEL_YUM_AUTOINSTALLUPDATES</label>
-        </field>
-
         <field
             type="literal"
             id="state_packagefunctions"
@@ -210,14 +203,6 @@
 
         <field
             type="select"
-            id="yum_AutoInstallUpdates"
-	    options="'disabled' => 'DISABLED', 'enabled' => 'ENABLED'"
-            value="get_status('AutoInstallUpdates')">
-            <label>LABEL_YUM_AUTOINSTALLUPDATES</label>
-        </field>
-
-        <field
-            type="select"
             id="yum_PackageFunctions"
 	    options="'disabled' => 'DISABLED', 'enabled' => 'ENABLED'"
             value="get_status('PackageFunctions')">
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm mezzanine_patched_smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm
--- smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm	2006-01-24 17:20:19.206678218 +1100
+++ mezzanine_patched_smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm	2006-01-24 17:20:02.969503304 +1100
@@ -180,7 +180,6 @@
 
     for my $param ( qw(
 			status 
-			AutoInstallUpdates
 			PackageFunctions
             	) )
     {
