diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.2/root/etc/e-smith/db/yum_repositories/migrate/20SMERepos mezzanine_patched_smeserver-yum-1.1.2/root/etc/e-smith/db/yum_repositories/migrate/20SMERepos
--- smeserver-yum-1.1.2/root/etc/e-smith/db/yum_repositories/migrate/20SMERepos	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.2/root/etc/e-smith/db/yum_repositories/migrate/20SMERepos	2006-03-06 16:53:36.185706728 +1100
@@ -0,0 +1,16 @@
+{
+    for my $name ( qw(addons dev os updates updates-testing) )
+    {
+	my $sme_name = "sme$name";
+
+	my $repo = $DB->get($sme_name) or next;
+
+	my $url = $repo->prop('BaseURL');
+	my $new_url = $url;
+
+	$new_url =~ s:/$name/:/$sme_name/:;
+	next if ($url eq $new_url);
+
+	$repo->set_prop('BaseURL', $new_url);
+    }
+}
