diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/GPGCheck	2005-09-25 15:35:10.267981768 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/GPGCheck	2005-09-25 15:34:14.689430992 -0600
@@ -1 +1 @@
-0
+no
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/GPGCheck	2005-09-25 15:35:10.268981616 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/GPGCheck	2005-09-25 15:34:18.077915864 -0600
@@ -1 +1 @@
-1
+yes
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosplus/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosplus/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosplus/GPGCheck	2005-09-25 15:35:10.268981616 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosplus/GPGCheck	2005-09-25 15:34:20.626528416 -0600
@@ -1 +1 @@
-0
+no
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/GPGCheck	2005-09-25 15:35:10.269981464 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/GPGCheck	2005-09-25 15:34:23.328117712 -0600
@@ -1 +1 @@
-0
+no
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/GPGCheck	2005-09-25 15:35:10.269981464 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/GPGCheck	2005-09-25 15:34:25.981714304 -0600
@@ -1 +1 @@
-0
+no
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/GPGCheck	2005-09-25 15:35:10.270981312 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/GPGCheck	2005-09-25 15:34:28.907269552 -0600
@@ -1 +1 @@
-0
+no
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/GPGCheck	2005-09-25 15:35:10.271981160 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/GPGCheck	2005-09-25 15:34:31.875818264 -0600
@@ -1 +1 @@
-0
+no
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/GPGCheck	2005-09-25 15:35:10.382964288 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/GPGCheck	2005-09-25 15:34:35.067333080 -0600
@@ -1 +1 @@
-1
+yes
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/GPGCheck	2005-09-25 15:35:10.272981008 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/GPGCheck	2005-09-25 15:34:37.834912344 -0600
@@ -1 +1 @@
-0
+no
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/GPGCheck	2005-09-25 15:35:10.272981008 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/GPGCheck	2005-09-25 15:34:41.769314224 -0600
@@ -1 +1 @@
-1
+yes
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/GPGCheck	2005-09-25 15:35:10.388963376 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/GPGCheck	2005-09-25 15:34:45.202792256 -0600
@@ -1 +1 @@
-1
+yes
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/GPGCheck	2005-09-25 15:35:10.273980856 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/GPGCheck	2005-09-25 15:34:48.110350240 -0600
@@ -1 +1 @@
-0
+no
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/GPGCheck	2005-09-25 15:35:10.273980856 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/GPGCheck	2005-09-25 15:34:53.020603768 -0600
@@ -1 +1 @@
-1
+yes
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/migrate/00openRW mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/migrate/00openRW
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/migrate/00openRW	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/migrate/00openRW	2005-09-25 15:33:26.419769096 -0600
@@ -0,0 +1,3 @@
+{
+    $DB = esmith::ConfigDB->open(${DB_FILENAME});
+}
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/migrate/10GPG_and_Groups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/migrate/10GPG_and_Groups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/migrate/10GPG_and_Groups	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/migrate/10GPG_and_Groups	2005-09-25 15:33:34.276574680 -0600
@@ -0,0 +1,14 @@
+{
+    for my $repo ($DB->get_all_by_prop(type => 'repository'))
+    {
+	%props = $repo->props;
+
+	$props{GPGCheck} = 'yes' if ($props{GPGCheck} =~ /1/);
+	$props{GPGCheck} = 'no'  if ($props{GPGCheck} =~ /0/);
+
+	$props{EnableGroups} = 'yes'  if ($props{EnableGroups} =~ /1/);
+	$props{EnableGroups} = 'no'   if ($props{EnableGroups} =~ /0/);
+
+	$repo->merge_props(%props);
+    }
+}
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/yum.conf/20repositories mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/yum.conf/20repositories
--- smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/yum.conf/20repositories	2005-09-25 15:35:10.411959880 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/yum.conf/20repositories	2005-09-25 15:19:47.055331424 -0600
@@ -12,11 +12,16 @@
 
 	$OUT .= "[" . $repos->key . "]\n";
 
-	for my $prop (qw(Name BaseURL GPGCheck EnableGroups))
+	for my $prop (qw(Name BaseURL))
 	{
 	    $OUT .= lc($prop) . '=' . $props{$prop} . "\n";
 	}
 
+	for my $prop (qw(GPGCheck EnableGroups))
+	{
+	    $OUT .= lc($prop) . '=' . ($props{$prop} eq "yes" ? 1 : 0) . "\n";
+	}
+
 	if (exists $props{Exclude})
 	{
 	    $OUT .= "exclude=" . join(' ', split(/,/, $props{Exclude})) . "\n";
