diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.1/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/yum mezzanine_patched_smeserver-yum-1.1.1/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/yum
--- smeserver-yum-1.1.1/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/yum	2005-09-25 20:30:54.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.1.1/root/etc/e-smith/locale/fr/etc/e-smith/web/functions/yum	2005-09-30 07:42:18.837112408 -0600
@@ -166,6 +166,9 @@
             <![CDATA[
 	    Les mises à jour suivantes sont installées 
 	    sur ce syst&egrave;me. 
+
+	    Vous ne devez retirez des paquetages que si vous êtes
+	    conscients des conséquences que cela peut engendrer.
 	    ]]>
 	</trans>
     </entry>
