diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.2/root/etc/e-smith/templates/etc/yum.conf/20repositories mezzanine_patched_smeserver-yum-1.1.2/root/etc/e-smith/templates/etc/yum.conf/20repositories
--- smeserver-yum-1.1.2/root/etc/e-smith/templates/etc/yum.conf/20repositories	2005-10-28 21:27:34.493649190 +1000
+++ mezzanine_patched_smeserver-yum-1.1.2/root/etc/e-smith/templates/etc/yum.conf/20repositories	2005-10-28 21:27:15.527872114 +1000
@@ -28,6 +28,11 @@
 	    $OUT .= "exclude=" . join(' ', split(/,/, $props{Exclude})) . "\n";
 	}	
 
+	if (exists $props{GPGKey})
+	{
+	    $OUT .= "gpgkey" . '=' . $props{GPGKey} . "\n";
+	} 
+
 	$OUT .= "\n";
     }
 }
