diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.2/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/yum mezzanine_patched_smeserver-yum-1.1.2/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/yum
--- smeserver-yum-1.1.2/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/yum	2006-02-16 16:46:17.111615283 +1100
+++ mezzanine_patched_smeserver-yum-1.1.2/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/yum	2006-02-16 16:42:06.000305830 +1100
@@ -260,4 +260,9 @@
 	<trans>Save</trans>
     </entry>
 
+    <entry>
+	<base>ERROR_UPDATING</base>
+	<trans>An error occurred while updating the system.</trans>
+    </entry>
+
 </lexicon>
