diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.2.0/createlinks mezzanine_patched_smeserver-yum-1.2.0/createlinks
--- smeserver-yum-1.2.0/createlinks	2008-01-07 04:34:53.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.2.0/createlinks	2008-01-07 04:34:11.000000000 -0700
@@ -26,12 +26,7 @@
     event_link("yum-import-keys", $_, "10");
 }
 
-# safe_symlink("once", "root/etc/e-smith/events/local/services2adjust/yum");
-
-for ( qw(local) )
-{
-    event_link("yum-update-dbs", $_, "90");
-}
+safe_symlink("once", "root/etc/e-smith/events/local/services2adjust/yum");
 
 #--------------------------------------------------
 # functions for manager panel
