diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/BaseURL	2005-07-13 22:43:54.425616632 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/BaseURL	2005-07-13 22:32:15.648976728 -0600
@@ -1 +1 @@
-http://mirror.contribs.org/smeserver/releases/7.0alpha/smeaddons/$basearch
+http://mirror.contribs.org/pub/smeserver/releases/7/addons/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/Name	2005-05-27 04:34:42.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/Name	2005-07-13 22:41:55.858270232 -0600
@@ -1 +1 @@
-SME Server 7 - smeaddons
+SME Server - addons
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/BaseURL	2005-07-13 22:43:54.426616480 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/BaseURL	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-http://mirror.contribs.org/smeserver/releases/7.0alpha/smecore/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/EnableGroups	2005-05-27 04:34:42.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/EnableGroups	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-1
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/GPGCheck	2005-07-13 22:43:54.818556896 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/GPGCheck	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-1
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/Name	2005-05-27 04:34:42.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/Name	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-SME Server 7 - smecore
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/status	2005-05-27 04:34:42.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/status	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-enabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/type	2005-05-27 04:34:42.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/type	1969-12-31 17:00:00.000000000 -0700
@@ -1 +0,0 @@
-repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/BaseURL	2005-07-13 22:43:54.428616176 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/BaseURL	2005-07-13 22:32:31.666541688 -0600
@@ -1 +1 @@
-http://mirror.contribs.org/smeserver/releases/7.0alpha/smedev/$basearch
+http://mirror.contribs.org/pub/smeserver/releases/7/dev/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/Name	2005-07-13 22:43:54.428616176 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/Name	2005-07-13 22:41:58.909806328 -0600
@@ -1 +1 @@
-SME Server 7 - smedev
+SME Server - dev
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/BaseURL	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/BaseURL	2005-07-13 22:32:24.598616176 -0600
@@ -0,0 +1 @@
+http://mirror.contribs.org/pub/smeserver/releases/7/os/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/EnableGroups	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/EnableGroups	2005-05-27 04:34:42.000000000 -0600
@@ -0,0 +1 @@
+1
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/GPGCheck	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/GPGCheck	2005-07-13 19:24:43.054702576 -0600
@@ -0,0 +1 @@
+1
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/Name	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/Name	2005-07-13 22:41:57.356042536 -0600
@@ -0,0 +1 @@
+SME Server - os
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/status	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/status	2005-05-27 04:34:42.000000000 -0600
@@ -0,0 +1 @@
+enabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/type	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeos/type	2005-05-27 04:34:42.000000000 -0600
@@ -0,0 +1 @@
+repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/BaseURL	2005-07-13 22:43:54.680577872 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/BaseURL	2005-07-13 22:32:36.234847200 -0600
@@ -1 +1 @@
-http://mirror.contribs.org/smeserver/releases/7.0alpha/smetest/$basearch
+http://mirror.contribs.org/pub/smeserver/releases/7/test/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/Name	2005-07-13 22:43:54.684577264 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/Name	2005-07-13 22:42:02.098321600 -0600
@@ -1 +1 @@
-SME Server 7 - smetest
+SME Server - test
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/BaseURL	2005-07-13 22:43:54.431615720 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/BaseURL	2005-07-13 22:32:43.186790344 -0600
@@ -1 +1 @@
-http://mirror.contribs.org/smeserver/releases/7.0alpha/smeupdates/$basearch
+http://mirror.contribs.org/pub/smeserver/releases/7/updates/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/Name	2005-05-27 04:34:42.000000000 -0600
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/Name	2005-07-13 22:42:03.684080528 -0600
@@ -1 +1 @@
-SME Server 7 - smeupdates
+SME Server - updates
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/BaseURL	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/BaseURL	2005-07-13 22:32:56.567756128 -0600
@@ -0,0 +1 @@
+http://mirror.contribs.org/pub/smeserver/releases/7/updates-testing/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/EnableGroups	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/EnableGroups	2005-07-13 22:32:48.592968480 -0600
@@ -0,0 +1 @@
+1
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/GPGCheck mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/GPGCheck
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/GPGCheck	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/GPGCheck	2005-07-13 22:32:48.593968328 -0600
@@ -0,0 +1 @@
+1
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/Name	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/Name	2005-07-13 22:42:19.101521544 -0600
@@ -0,0 +1 @@
+SME Server - updates testing
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/status	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/status	2005-07-13 22:33:58.135166920 -0600
@@ -0,0 +1 @@
+disabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/type	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates-testing/type	2005-07-13 22:33:55.083630824 -0600
@@ -0,0 +1 @@
+repository
