diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.2/root/sbin/e-smith/yum mezzanine_patched_smeserver-yum-1.1.2/root/sbin/e-smith/yum
--- smeserver-yum-1.1.2/root/sbin/e-smith/yum	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.2/root/sbin/e-smith/yum	2006-01-24 17:52:12.919713655 +1100
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+echo "==========================================================="
+echo "WARNING: Additional commands are required after running yum"
+echo "==========================================================="
+
+/usr/bin/yum $@
+status=$?
+
+echo "==========================================================="
+echo "WARNING: You now need to run BOTH of the following comands"
+echo
+echo "signal-event post-upgrade; signal-event reboot"
+echo "=========================================================="
+
+exit $status
