diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/BaseURL	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/BaseURL	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+http://mirror.centos.org/centos/3/addons/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/EnableGroups	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/EnableGroups	2005-05-31 12:40:21.000000000 +1000
@@ -0,0 +1 @@
+0
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/Name	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/Name	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+CentOS 3 - addons
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/status	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/status	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+disabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/type	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/addons/type	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/BaseURL	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/BaseURL	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+http://mirror.centos.org/centos/3/os/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/EnableGroups	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/EnableGroups	2005-05-31 12:40:21.000000000 +1000
@@ -0,0 +1 @@
+0
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/Name	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/Name	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+CentOS 3 - os
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/status	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/status	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+enabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/type	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/base/type	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/BaseURL	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/BaseURL	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-http://mirror.centos.org/centos/3/addons/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/EnableGroups	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/EnableGroups	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-0
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/Name	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/Name	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-CentOS 3 - addons
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/Order mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/Order
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/Order	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/Order	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-35
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/status	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/status	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-disabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/type	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosaddons/type	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/BaseURL	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/BaseURL	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-http://mirror.centos.org/centos/3/contrib/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/EnableGroups	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/EnableGroups	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-0
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/Name	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/Name	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-CentOS 3 - contrib
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/Order mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/Order
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/Order	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/Order	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-35
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/status	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/status	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-enabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/type	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centoscontrib/type	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/BaseURL	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/BaseURL	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-http://mirror.centos.org/centos/3/extras/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/EnableGroups	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/EnableGroups	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-0
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/Name	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/Name	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-CentOS 3 - extras
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/Order mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/Order
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/Order	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/Order	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-35
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/status	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/status	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-disabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/type	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosextras/type	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/BaseURL	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/BaseURL	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-http://mirror.centos.org/centos/3/os/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/EnableGroups	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/EnableGroups	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-0
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/Name	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/Name	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-CentOS 3 - os
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/Order mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/Order
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/Order	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/Order	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-30
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/status	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/status	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-enabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/type	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosos/type	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosplus/Order mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosplus/Order
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosplus/Order	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosplus/Order	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-35
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/BaseURL	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/BaseURL	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-http://mirror.centos.org/centos/3/testing/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/EnableGroups	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/EnableGroups	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-0
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/Name	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/Name	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-CentOS 3 - testing
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/Order mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/Order
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/Order	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/Order	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-40
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/status	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/status	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-disabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/type	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centostesting/type	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/BaseURL	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/BaseURL	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-http://mirror.centos.org/centos/3/updates/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/EnableGroups	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/EnableGroups	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-0
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/Name	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/Name	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-CentOS 3 - updates
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/Order mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/Order
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/Order	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/Order	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-30
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/status	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/status	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-enabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/type	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/centosupdates/type	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/BaseURL	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/BaseURL	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+http://mirror.centos.org/centos/3/contrib/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/EnableGroups	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/EnableGroups	2005-05-31 12:40:21.000000000 +1000
@@ -0,0 +1 @@
+0
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/Name	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/Name	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+CentOS 3 - contrib
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/status	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/status	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+enabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/type	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/contrib/type	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/BaseURL	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/BaseURL	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+http://mirror.centos.org/centos/3/extras/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/EnableGroups	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/EnableGroups	2005-05-31 12:40:21.000000000 +1000
@@ -0,0 +1 @@
+0
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/Name	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/Name	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+CentOS 3 - extras
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/status	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/status	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+disabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/type	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/extras/type	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/Order mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/Order
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/Order	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeaddons/Order	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-65
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/Order mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/Order
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/Order	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smecore/Order	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-60
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/Order mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/Order
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/Order	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smedev/Order	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-70
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/BaseURL	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/BaseURL	2005-05-31 12:42:01.000000000 +1000
@@ -0,0 +1 @@
+http://mirror.contribs.org/smeserver/releases/7.0alpha/smetest/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/EnableGroups	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/EnableGroups	2005-05-31 12:40:21.000000000 +1000
@@ -0,0 +1 @@
+1
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/Name	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/Name	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+SME Server 7 - smetest
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/status	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/status	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+disabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/type	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetest/type	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/BaseURL	2005-06-03 11:20:45.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/BaseURL	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-http://mirror.contribs.org/smeserver/releases/7.0alpha/smetest/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/EnableGroups	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/EnableGroups	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-1
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/Name	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/Name	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-SME Server 7 - smetest
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/Order mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/Order
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/Order	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/Order	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-70
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/status	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/status	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-disabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/type	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smetesting/type	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/Order mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/Order
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/Order	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/smeupdates/Order	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-60
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/BaseURL	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/BaseURL	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+http://mirror.centos.org/centos/3/testing/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/EnableGroups	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/EnableGroups	2005-05-31 12:40:21.000000000 +1000
@@ -0,0 +1 @@
+0
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/Name	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/Name	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+CentOS 3 - testing
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/status	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/status	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+disabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/type	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/testing/type	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/BaseURL mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/BaseURL
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/BaseURL	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/BaseURL	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+http://mirror.centos.org/centos/3/updates/$basearch
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/EnableGroups mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/EnableGroups
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/EnableGroups	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/EnableGroups	2005-05-31 12:40:21.000000000 +1000
@@ -0,0 +1 @@
+0
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/Name mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/Name
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/Name	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/Name	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+CentOS 3 - updates
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/status mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/status
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/status	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/status	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+enabled
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/type mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/type
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/type	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/defaults/updates/type	2005-05-27 20:34:42.000000000 +1000
@@ -0,0 +1 @@
+repository
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/force/base/Exclude mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/force/base/Exclude
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/force/base/Exclude	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/force/base/Exclude	2005-06-03 11:10:20.000000000 +1000
@@ -0,0 +1 @@
+dump,stunnel
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/force/updates/Exclude mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/force/updates/Exclude
--- smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/force/updates/Exclude	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/db/yum_repositories/force/updates/Exclude	2005-06-03 11:10:14.000000000 +1000
@@ -0,0 +1 @@
+dump,stunnel
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/yum.conf/10main_pkgpolicy mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/yum.conf/10main_pkgpolicy
--- smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/yum.conf/10main_pkgpolicy	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/yum.conf/10main_pkgpolicy	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-pkgpolicy=last
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/yum.conf/20repositories mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/yum.conf/20repositories
--- smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/yum.conf/20repositories	2005-05-27 20:34:42.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/templates/etc/yum.conf/20repositories	2005-06-03 11:18:06.000000000 +1000
@@ -4,20 +4,24 @@
 
     $OUT = "\n";
 
-    for my $repos ( sort { $a->prop('Order') <=> $b->prop('Order') } 
-			$ydb->get_all_by_prop('type' => 'repository') )
+    for my $repos ( $ydb->get_all_by_prop('type' => 'repository') )
     {
         next unless ($repos->prop('status') eq 'enabled');
 
 	my %props = ($DB->get('yum')->props(), $repos->props());
 
-	$OUT .= sprintf "[%03d%s]\n", $props{Order}, $repos->key;
+	$OUT .= "[" . $repos->key . "]\n";
 
 	for my $prop (qw(Name BaseURL GPGCheck EnableGroups))
 	{
 	    $OUT .= lc($prop) . '=' . $props{$prop} . "\n";
 	}
 
+	if (exists $props{Exclude})
+	{
+	    $OUT .= "exclude=" . join(' ', split(/,/, $props{Exclude})) . "\n";
+	}	
+
 	$OUT .= "\n";
     }
 }
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm mezzanine_patched_smeserver-yum-1.1.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm
--- smeserver-yum-1.1.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm	2005-06-03 11:20:45.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm	2005-06-03 11:19:28.000000000 +1000
@@ -35,9 +35,7 @@
 
 our @ISA = qw(esmith::FormMagick Exporter);
 
-our @EXPORT = qw(
-        get_prop get_value 
-        );
+our @EXPORT = qw();
 
 our $VERSION = sprintf '%d.%03d', q$Revision: 1.0 $ =~ /: (\d+).(\d+)/;
 our $db = esmith::ConfigDB->open or
