diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm mezzanine_patched_smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm
--- smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm	2006-01-24 18:12:38.167534302 +1100
+++ mezzanine_patched_smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm	2006-01-24 18:12:20.156667991 +1100
@@ -2,7 +2,7 @@
 
 #----------------------------------------------------------------------
 # copyright (C) 2004 Shad L. Lords <slords@mail.com>
-# Copyright (C) 2005 Gordon Rowell <gordonr@gormand.com.au>
+# Copyright (C) 2005-2006 Gordon Rowell <gordonr@gormand.com.au>
 # Copyright (C) 2005 Darrell May <dmay@myezserver.com>
 # 
 # This program is free software; you can redistribute it and/or modify
@@ -282,6 +282,12 @@
 	return undef;
     }
 
+    unless (system( "/sbin/e-smith/signal-event", "reboot") == 0)
+    {
+	$fm->error('ERROR_UPDATING');
+	return undef;
+    }
+
     $fm->success('SUCCESS');
 }
 1;
