diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/etc/e-smith/web/functions/yum mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/web/functions/yum
--- smeserver-yum-1.1.0/root/etc/e-smith/web/functions/yum	2005-05-31 13:10:12.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/etc/e-smith/web/functions/yum	2005-05-31 13:03:59.000000000 +1000
@@ -148,6 +148,7 @@
         <field
             type="select" multiple="1"
             id="SelectedGroups"
+	    display="non_empty('available','group')"
             options="get_options('available','group')">
             <label>LABEL_AVAILABLE_GROUPS</label>
             <description>DESC_AVAILABLE_GROUPS</description>
@@ -156,7 +157,7 @@
         <field
             type="select" multiple="1"
             id="SelectedPackages"
-	    display="package_functions_enabled()"
+	    display="non_empty('available','packages')"
             options="get_options('available','package')">
             <label>LABEL_AVAILABLE_PACKAGES</label>
             <description>DESC_AVAILABLE_PACKAGES</description>
@@ -177,6 +178,7 @@
         <field
             type="select" multiple="1"
             id="SelectedGroups"
+	    display="non_empty('installed','group')"
             options="get_options('installed','group')">
             <label>LABEL_INSTALLED_GROUPS</label>
             <description>DESC_INSTALLED_GROUPS</description>
@@ -185,7 +187,7 @@
         <field
             type="select" multiple="1"
             id="SelectedPackages"
-	    display="package_functions_enabled()"
+	    display="non_empty('installed','package')"
             options="get_options('installed','package')">
             <label>LABEL_INSTALLED_PACKAGES</label>
             <description>DESC_INSTALLED_PACKAGES</description>
diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm mezzanine_patched_smeserver-yum-1.1.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm
--- smeserver-yum-1.1.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm	2005-05-31 13:10:12.000000000 +1000
+++ mezzanine_patched_smeserver-yum-1.1.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm	2005-05-31 13:09:39.000000000 +1000
@@ -112,12 +112,20 @@
 
 sub non_empty
 {
-    my ($fm, $yumdb) = @_;
+    my ($fm, $yumdb, $type) = @_;
+
+    $type ||= 'both';
 
     return 0 unless (exists $dbs{$yumdb});
 
     my $groups   = scalar $dbs{$yumdb}->get_all_by_prop(type => 'group');
+    return $groups if ($type eq 'group');
+
     my $packages = scalar $dbs{$yumdb}->get_all_by_prop(type => 'package');
+    if ($type eq 'package')
+    {
+	return $fm->package_functions_enabled ? $packages : 0;
+    }
 
     return ($fm->package_functions_enabled or $yumdb eq 'updates') ?
 		($groups || $packages) : $groups;
