diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.1/root/etc/e-smith/locale/it/etc/e-smith/web/functions/yum mezzanine_patched_smeserver-yum-1.1.1/root/etc/e-smith/locale/it/etc/e-smith/web/functions/yum
--- smeserver-yum-1.1.1/root/etc/e-smith/locale/it/etc/e-smith/web/functions/yum	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_smeserver-yum-1.1.1/root/etc/e-smith/locale/it/etc/e-smith/web/functions/yum	2005-09-30 06:07:15.072216472 -0600
@@ -0,0 +1,264 @@
+<lexicon lang="it"> 
+    <entry>
+        <base>FORM_TITLE</base>
+        <trans>Installazione Software</trans>
+    </entry>
+
+    <entry>
+	<base>Configuration</base>
+	<trans>Configurazione</trans>
+    </entry>
+
+    <entry>
+	<base>Software installer</base>
+	<trans>Installazione Software</trans>
+    </entry>
+
+    <entry>
+        <base>SUCCESS</base>
+        <trans>La nuova configurazione è stata salvata.</trans>
+    </entry>
+
+    <entry>
+        <base>LABEL_YUM_STATUS</base>
+        <trans>
+            Verifica presenza aggiornamenti ogni giorno
+        </trans>
+    </entry>
+
+    <entry>
+	<base>LABEL_YUM_AUTOINSTALLUPDATES</base>
+	<trans>Installazione automatica aggiormanenti</trans>
+   </entry>
+
+    <entry>
+	<base>LABEL_YUM_PACKAGEFUNCTIONS</base>
+	<trans>Gestione dei singoli pacchetti</trans>
+    </entry>
+
+    <entry>
+	<base>DESC_YUM_PACKAGEFUNCTIONS</base>
+	<trans>
+        Installazione software può gestire gruppi di 
+        pacchetti o pacchetti singoli.
+        Di default, vengono gestiti solo gruppi di pacchetti.
+        Se si desidera gestire pacchetti singoli abilitare questa opzione.
+        L'opzione andrebbe usata solo da amministratori esperti.
+	</trans>
+    </entry>
+
+    <entry>
+	<base>LABEL_ENABLED_REPOSITORIES</base>
+	<trans>repositories abilitati</trans>
+    </entry>
+
+    <entry>
+	<base>DESC_ENABLED_REPOSITORIES</base>
+	<trans>
+        Installazione software può installare pacchetti 
+        da diversi repository.
+        Per abilitare un repository, selezionarlo dalla lista.
+        Per dibasilitare deselezionarlo dalla lista.
+        Di default, solo il sofware stabile è disponibile per
+        l'installazione.
+	</trans>
+    </entry>
+
+    <entry>
+        <base>DESC_UPDATES_AVAILABLE</base>
+        <trans>
+            <![CDATA[
+            <div class="success">
+                <h2>Aggiornamenti disponibili</h2>
+                Ci sono aggiornamenti disponibili per il sistema.
+                Installarli appena possibile.
+            </div>
+            ]]>
+        </trans>
+    </entry>
+
+    <entry>
+        <base>LABEL_AVAILABLE_UPDATES</base>
+        <trans>Aggiornamenti disponibili</trans>
+    </entry>
+
+    <entry>
+        <base>DESC_AVAILABLE_UPDATES</base>
+        <trans>
+            Di seguito sono riportati gli aggiornamenti disponibili.
+            Normalmente tutti gli aggiornamenti disponibili dovranno
+            esssere installati.
+            Se ci sono degli aggiornamenti che non si desidera installare,
+            deselezionarli dalla lista sottostante.
+        </trans>
+    </entry>
+
+    <entry>
+        <base>INSTALL_UPDATES</base>
+        <trans>Installa aggiornamenti selezionati</trans>
+    </entry>
+
+    <entry>
+        <base>DESC_BUTTON_INSTALL_UPDATES</base>
+        <trans>
+            <![CDATA[
+            <a class="button-like"
+                href="yum?page=0&page_stack=&Next=Next&wherenext=YUM_PAGE_INSTALL_UPDATES">
+                Installa aggiornamenti disponibili</a>
+                ]]>
+        </trans>
+    </entry>
+
+    <entry>
+	<base>HEADER_AVAILABLE_SOFTWARE</base>
+	<trans>
+        E' possibile installare il seguente software addizionale.
+        Non installare software addizionale se non si è sicuri di
+        quello che si sta facendo.
+	</trans>
+    </entry>
+
+    <entry>
+	<base>DESC_AVAILABLE_GROUPS</base>
+    <trans>
+        E' possibile selezionare gruppi di software
+        da installare dalla lista sottostante.
+	</trans>
+    </entry>
+
+    <entry>
+	<base>LABEL_AVAILABLE_GROUPS</base>
+	<trans>Gruppi disponibili</trans>
+    </entry>
+
+    <entry>
+	<base>DESC_AVAILABLE_PACKAGES</base>
+    <trans>
+        E' possibile selezionare pacchetti individuali
+        da installare dalla lista sottostante.
+	</trans>
+    </entry>
+
+    <entry>
+	<base>LABEL_AVAILABLE_PACKAGES</base>
+	<trans>Pacchetti disponibili</trans>
+    </entry>
+
+    <entry>
+	<base>INSTALL_SOFTWARE</base>
+	<trans>Installa</trans>
+    </entry>
+
+    <entry>
+	<base>HEADER_INSTALLED_SOFTWARE</base>
+	<trans>
+        Elenco dei pacchetti installati nel sistema.
+        Non rimuovere pacchetti se non si è sicuri di
+        quello che si sta facendo.
+	</trans>
+    </entry>
+
+    <entry>
+	<base>DESC_INSTALLED_GROUPS</base>
+	<trans>
+        E' possibile selezionare gruppi
+        da rimuovere dalla lista sottostante.
+	</trans>
+    </entry>
+
+    <entry>
+	<base>LABEL_INSTALLED_GROUPS</base>
+	<trans>Gruppi installati</trans>
+    </entry>
+
+    <entry>
+	<base>DESC_INSTALLED_PACKAGES</base>
+    <trans>
+        E' possibile selezionare pacchetti singoli 
+        da rimuovere dalla lista sottostante.
+	</trans>
+    </entry>
+
+    <entry>
+	<base>LABEL_INSTALLED_PACKAGES</base>
+	<trans>Pacchetti installati</trans>
+    </entry>
+
+    <entry>
+	<base>REMOVE_SOFTWARE</base>
+	<trans>Rimuovi</trans>
+    </entry>
+
+    <entry>
+        <base>DESC_BUTTON_CONFIGURATION</base>
+        <trans>
+            <![CDATA[
+            <a class="button-like"
+                href="yum?page=0&page_stack=&Next=Next&wherenext=YUM_PAGE_CONFIGURATION">
+                           Cambia impostazioni installazione software</a>
+                ]]>
+        </trans>
+    </entry>
+
+    <entry>
+        <base>DESC_BUTTON_INSTALL_AVAILABLE</base>
+        <trans>
+            <![CDATA[
+            <a class="button-like"
+                href="yum?page=0&page_stack=&Next=Next&wherenext=YUM_PAGE_INSTALL_AVAILABLE">
+                Installa software addizionale </a>
+                ]]>
+	</trans>
+    </entry>
+
+    <entry>
+        <base>DESC_BUTTON_REMOVE</base>
+        <trans>
+            <![CDATA[
+            <a class="button-like"
+                href="yum?page=0&page_stack=&Next=Next&wherenext=YUM_PAGE_REMOVE">
+                Rimuovi software installato</a>
+                ]]>
+	</trans>
+    </entry>
+
+    <entry>
+        <base>DESC_SECTIONBAR</base>
+        <trans>
+            <![CDATA[
+            <hr class="sectionbar"/>
+            ]]>
+        </trans>
+    </entry>
+
+    <entry>
+	<base>HEADER_POSTUPGRADE_REQUIRED</base>
+	<trans>
+        Il sistema deve essere riconfigurato dopo 
+        aver aggiunto o rimosso software o applicato aggiornamenti.
+        Terminato il processo di riconfigurazione è neccessario
+        riavviare il sistema.
+	</trans>
+     </entry>
+
+    <entry>
+	<base>POST_UPGRADE</base>
+	<trans>Riconfigurazione</trans>
+    </entry>
+
+    <entry>
+	<base>ENABLED</base>
+	<trans>Abilita</trans>
+    </entry>
+
+    <entry>
+	<base>DISABLED</base>
+	<trans>Disailita</trans>
+    </entry>
+
+    <entry>
+	<base>SAVE</base>
+	<trans>Salva</trans>
+    </entry>
+
+</lexicon>
