diff -Nur -x '*.orig' -x '*.rej' smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm mezzanine_patched_smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm
--- smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm	2005-11-08 12:42:02.153550163 +1100
+++ mezzanine_patched_smeserver-yum-1.1.2/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/yum.pm	2005-11-08 12:40:05.026859860 +1100
@@ -224,10 +224,7 @@
 	return undef;
     }
 
-    my $page = ($function eq 'modify') ? 'First' 
-				       : 'YUM_PAGE_POSTUPGRADE';
-
-    $fm->success('SUCCESS', $page);
+    $fm->success('SUCCESS', 'First');
 }
 
 sub package_functions_enabled
